+++
# A Demo section created with the Blank widget.
# Any elements can be added in the body: https://sourcethemes.com/academic/docs/writing-markdown-latex/
# Add more sections by duplicating this file and customizing to your requirements.

widget = "blank"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 10  # Order that this section will appear.

title = "Genetics"
subtitle = ""

[design]
  # Choose how many columns the section has. Valid values: 1 or 2.
  columns = "1"

[design.background]
  # Apply a background color, gradient, or image.
  #   Uncomment (by removing `#`) an option to apply it.
  #   Choose a light or dark text color by setting `text_color_light`.
  #   Any HTML color name or Hex value is valid.

  # Background color.
  # color = "navy"
  
  # Background gradient.
  # gradient_start = "DarkGreen"
  # gradient_end = "ForestGreen"
  
  # Background image.
  image = "louis-reed-pwcKF7L4-no-unsplash.jpg"  # Name of image in `static/img/`.
  image_darken = 0.6  # Darken the image? Range 0-1 where 0 is transparent and 1 is opaque.
  # image_size = "cover"  #  Options are `cover` (default), `contain`, or `actual` size.
  # image_position = "center"  # Options include `left`, `center` (default), or `right`.
  # image_parallax = true  # Use a fun parallax-like fixed background effect? true/false
  
  # Text color (true=light or false=dark).
  text_color_light = true

[design.spacing]
  # Customize the section spacing. Order is top, right, bottom, left.
  padding = ["100px", "0", "100px", "0"]

[advanced]
 # Custom CSS. 
 css_style = ""
 
 # CSS class.
 css_class = ""
+++
Proteomics approaches can identify physical interactions between proteins, but they do not necessarily reveal pathway-level relationships. Genetic screens, where the growth of a double mutant is compared to the fitness of the corresponding single mutants, can elucidate the functional relationships between genes. 

Inspired by successes in other model organisms, in collaboration with Emili and Greenblatt groups, we recently developed a high-throughput genetic screening approach for prokaryotes, termed eSGA (E. coli Synthetic Genetic Arrays). We used this eSGA approach to generate a high-density functional association map for genes involved in cell envelope maintenance or assembly. 

We are currently focusing on generating large-scale genetic interaction maps for essential biological processes using hypomorphic alleles, combining genetics and drug screening data to elucidate novel drug targets, and developing an advanced analogous genetic interaction technology for other prokaryotes and eukaryotes, including CRISPR-based methodologies for pathway analysis in human neuron models for gene variants associated with neurodegenerative diseases.