<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Dr. Mohan Babu">

  
  
  
    
  
  <meta name="description" content="Cellular processes are carried out through a series of molecular interactions. Various experimental approaches can be used to investigate these functional relationships on a large-scale. Recently, the power of investigating biological systems from the perspective of genetic (gene-gene or epistatic) interactions has been evidenced by the ability to elucidate novel functional relationships. Examples of functionally related genes include genes that buffer each other&#39;s function or impinge on the same biological process. Genetic interactions have traditionally been investigated in bacteria by combining pairs of mutations (e.g., gene deletions) and assessing deviation of the phenotype of each double mutant from an expected neutral (or no interaction) phenotype. Fitness is a particularly convenient phenotype to measure: when the double mutant grows faster or slower than expected, the two mutated genes are said to show alleviating or aggravating interactions, respectively. The most commonly used neutral model assumes that the fitness of the double mutant is equal to the product of individual single mutant fitness. A striking genetic interaction is exemplified by the loss of two nonessential genes that buffer each other in performing an essential biological function: deleting only one of these genes produces no detectable fitness defect; however, loss of both genes simultaneously results in systems failure, leading to synthetic sickness or lethality. Systematic large-scale genetic interaction screens have been used to generate functional maps for model eukaryotic organisms, such as yeast, to describe the functional organization of gene products into pathways and protein complexes within a cell. They also reveal the modular arrangement and cross talk of pathways and complexes within broader functional neighborhoods (Dixon et al., Annu Rev Genet 43:601-625, 2009). Here, we present a high-throughput quantitative Escherichia coli Synthetic Genetic Array (eSGA) screening procedure, which we developed to systematically infer genetic interactions by scoring growth defects among large numbers of double mutants in a classic Gram-negative bacterium. The eSGA method exploits the rapid colony growth, ease of genetic manipulation, and natural efficient genetic exchange via conjugation of laboratory E. coli strains. Replica pinning is used to grow and mate arrayed sets of single gene mutant strains and to select double mutants en masse. Strain fitness, which is used as the eSGA readout, is quantified by the digital imaging of the plates and subsequent measuring and comparing single and double mutant colony sizes. While eSGA can be used to screen select mutants to probe the functions of individual genes, using eSGA more broadly to collect genetic interaction data for many combinations of genes can help reconstruct a functional interaction network to reveal novel links and components of biological pathways as well as unexpected connections between pathways. A variety of bacterial systems can be investigated, wherein the genes impinge on a essential biological process (e.g., cell wall assembly, ribosome biogenesis, chromosome replication) that are of interest from the perspective of drug development (Babu et al., Mol Biosyst 12:1439-1455, 2009). We also show how genetic interactions generated by high-throughput eSGA screens can be validated by manual small-scale genetic crosses and by genetic complementation and gene rescue experiments.">

  
  <link rel="alternate" hreflang="en-us" href="http://192.157.236.170/BabuLab/publication/pmid-21815091/">

  


  
  
  
  <meta name="theme-color" content="#085757">
  

  
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css" crossorigin="anonymous" title="hl-light" disabled>
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark">
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css" integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin="anonymous">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/BabuLab/css/academic.css">

  




  


  
  

  

  <link rel="manifest" href="/BabuLab/index.webmanifest">
  <link rel="icon" type="image/png" href="/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="http://192.157.236.170/BabuLab/publication/pmid-21815091/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="Babu Lab">
  <meta property="og:url" content="http://192.157.236.170/BabuLab/publication/pmid-21815091/">
  <meta property="og:title" content="Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli. | Babu Lab">
  <meta property="og:description" content="Cellular processes are carried out through a series of molecular interactions. Various experimental approaches can be used to investigate these functional relationships on a large-scale. Recently, the power of investigating biological systems from the perspective of genetic (gene-gene or epistatic) interactions has been evidenced by the ability to elucidate novel functional relationships. Examples of functionally related genes include genes that buffer each other&#39;s function or impinge on the same biological process. Genetic interactions have traditionally been investigated in bacteria by combining pairs of mutations (e.g., gene deletions) and assessing deviation of the phenotype of each double mutant from an expected neutral (or no interaction) phenotype. Fitness is a particularly convenient phenotype to measure: when the double mutant grows faster or slower than expected, the two mutated genes are said to show alleviating or aggravating interactions, respectively. The most commonly used neutral model assumes that the fitness of the double mutant is equal to the product of individual single mutant fitness. A striking genetic interaction is exemplified by the loss of two nonessential genes that buffer each other in performing an essential biological function: deleting only one of these genes produces no detectable fitness defect; however, loss of both genes simultaneously results in systems failure, leading to synthetic sickness or lethality. Systematic large-scale genetic interaction screens have been used to generate functional maps for model eukaryotic organisms, such as yeast, to describe the functional organization of gene products into pathways and protein complexes within a cell. They also reveal the modular arrangement and cross talk of pathways and complexes within broader functional neighborhoods (Dixon et al., Annu Rev Genet 43:601-625, 2009). Here, we present a high-throughput quantitative Escherichia coli Synthetic Genetic Array (eSGA) screening procedure, which we developed to systematically infer genetic interactions by scoring growth defects among large numbers of double mutants in a classic Gram-negative bacterium. The eSGA method exploits the rapid colony growth, ease of genetic manipulation, and natural efficient genetic exchange via conjugation of laboratory E. coli strains. Replica pinning is used to grow and mate arrayed sets of single gene mutant strains and to select double mutants en masse. Strain fitness, which is used as the eSGA readout, is quantified by the digital imaging of the plates and subsequent measuring and comparing single and double mutant colony sizes. While eSGA can be used to screen select mutants to probe the functions of individual genes, using eSGA more broadly to collect genetic interaction data for many combinations of genes can help reconstruct a functional interaction network to reveal novel links and components of biological pathways as well as unexpected connections between pathways. A variety of bacterial systems can be investigated, wherein the genes impinge on a essential biological process (e.g., cell wall assembly, ribosome biogenesis, chromosome replication) that are of interest from the perspective of drug development (Babu et al., Mol Biosyst 12:1439-1455, 2009). We also show how genetic interactions generated by high-throughput eSGA screens can be validated by manual small-scale genetic crosses and by genetic complementation and gene rescue experiments."><meta property="og:image" content="http://192.157.236.170/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="http://192.157.236.170/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2020-07-10T06:21:58&#43;00:00">
    
    <meta property="article:modified_time" content="2011-01-01T00:00:00&#43;00:00">
  

  


    









<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "http://192.157.236.170/BabuLab/publication/pmid-21815091/"
  },
  "headline": "Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli.",
  
  "datePublished": "2020-07-10T06:21:58Z",
  "dateModified": "2011-01-01T00:00:00Z",
  
  "author": {
    "@type": "Person",
    "name": "Mohan Babu"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "Babu Lab",
    "logo": {
      "@type": "ImageObject",
      "url": "http://192.157.236.170/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_192x192_fill_lanczos_center_2.png"
    }
  },
  "description": "Cellular processes are carried out through a series of molecular interactions. Various experimental approaches can be used to investigate these functional relationships on a large-scale. Recently, the power of investigating biological systems from the perspective of genetic (gene-gene or epistatic) interactions has been evidenced by the ability to elucidate novel functional relationships. Examples of functionally related genes include genes that buffer each other's function or impinge on the same biological process. Genetic interactions have traditionally been investigated in bacteria by combining pairs of mutations (e.g., gene deletions) and assessing deviation of the phenotype of each double mutant from an expected neutral (or no interaction) phenotype. Fitness is a particularly convenient phenotype to measure: when the double mutant grows faster or slower than expected, the two mutated genes are said to show alleviating or aggravating interactions, respectively. The most commonly used neutral model assumes that the fitness of the double mutant is equal to the product of individual single mutant fitness. A striking genetic interaction is exemplified by the loss of two nonessential genes that buffer each other in performing an essential biological function: deleting only one of these genes produces no detectable fitness defect; however, loss of both genes simultaneously results in systems failure, leading to synthetic sickness or lethality. Systematic large-scale genetic interaction screens have been used to generate functional maps for model eukaryotic organisms, such as yeast, to describe the functional organization of gene products into pathways and protein complexes within a cell. They also reveal the modular arrangement and cross talk of pathways and complexes within broader functional neighborhoods (Dixon et al., Annu Rev Genet 43:601-625, 2009). Here, we present a high-throughput quantitative Escherichia coli Synthetic Genetic Array (eSGA) screening procedure, which we developed to systematically infer genetic interactions by scoring growth defects among large numbers of double mutants in a classic Gram-negative bacterium. The eSGA method exploits the rapid colony growth, ease of genetic manipulation, and natural efficient genetic exchange via conjugation of laboratory E. coli strains. Replica pinning is used to grow and mate arrayed sets of single gene mutant strains and to select double mutants en masse. Strain fitness, which is used as the eSGA readout, is quantified by the digital imaging of the plates and subsequent measuring and comparing single and double mutant colony sizes. While eSGA can be used to screen select mutants to probe the functions of individual genes, using eSGA more broadly to collect genetic interaction data for many combinations of genes can help reconstruct a functional interaction network to reveal novel links and components of biological pathways as well as unexpected connections between pathways. A variety of bacterial systems can be investigated, wherein the genes impinge on a essential biological process (e.g., cell wall assembly, ribosome biogenesis, chromosome replication) that are of interest from the perspective of drug development (Babu et al., Mol Biosyst 12:1439-1455, 2009). We also show how genetic interactions generated by high-throughput eSGA screens can be validated by manual small-scale genetic crosses and by genetic complementation and gene rescue experiments."
}
</script>

  

  


  


  





  <title>Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli. | Babu Lab</title>

</head>
<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="dark">

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  









<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/BabuLab/">Babu Lab</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/BabuLab/">Babu Lab</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/labnews/"><span>News</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/labmember/"><span>Members</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link  active" href="/BabuLab/publication/"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/labresource/"><span>Resources</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/pgcf/"><span>PGCF</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/#contact"><span>Contact</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link js-theme-selector" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-palette" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>


  <div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli.</h1>

  

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Alla Gagarinova</span>, <span >Jack Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2011
  </span>
  

  

  

  
  
  

  
  

</div>

    











  



<div class="btn-links mb-3">
  
  








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 js-cite-modal"
        data-filename="/BabuLab/publication/pmid-21815091/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1" href="https://doi.org/10.1007/978-1-61779-197-0_9" target="_blank" rel="noopener">
  DOI
</a>



</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">Cellular processes are carried out through a series of molecular interactions. Various experimental approaches can be used to investigate these functional relationships on a large-scale. Recently, the power of investigating biological systems from the perspective of genetic (gene-gene or epistatic) interactions has been evidenced by the ability to elucidate novel functional relationships. Examples of functionally related genes include genes that buffer each other&rsquo;s function or impinge on the same biological process. Genetic interactions have traditionally been investigated in bacteria by combining pairs of mutations (e.g., gene deletions) and assessing deviation of the phenotype of each double mutant from an expected neutral (or no interaction) phenotype. Fitness is a particularly convenient phenotype to measure: when the double mutant grows faster or slower than expected, the two mutated genes are said to show alleviating or aggravating interactions, respectively. The most commonly used neutral model assumes that the fitness of the double mutant is equal to the product of individual single mutant fitness. A striking genetic interaction is exemplified by the loss of two nonessential genes that buffer each other in performing an essential biological function: deleting only one of these genes produces no detectable fitness defect; however, loss of both genes simultaneously results in systems failure, leading to synthetic sickness or lethality. Systematic large-scale genetic interaction screens have been used to generate functional maps for model eukaryotic organisms, such as yeast, to describe the functional organization of gene products into pathways and protein complexes within a cell. They also reveal the modular arrangement and cross talk of pathways and complexes within broader functional neighborhoods (Dixon et al., Annu Rev Genet 43:601-625, 2009). Here, we present a high-throughput quantitative Escherichia coli Synthetic Genetic Array (eSGA) screening procedure, which we developed to systematically infer genetic interactions by scoring growth defects among large numbers of double mutants in a classic Gram-negative bacterium. The eSGA method exploits the rapid colony growth, ease of genetic manipulation, and natural efficient genetic exchange via conjugation of laboratory E. coli strains. Replica pinning is used to grow and mate arrayed sets of single gene mutant strains and to select double mutants en masse. Strain fitness, which is used as the eSGA readout, is quantified by the digital imaging of the plates and subsequent measuring and comparing single and double mutant colony sizes. While eSGA can be used to screen select mutants to probe the functions of individual genes, using eSGA more broadly to collect genetic interaction data for many combinations of genes can help reconstruct a functional interaction network to reveal novel links and components of biological pathways as well as unexpected connections between pathways. A variety of bacterial systems can be investigated, wherein the genes impinge on a essential biological process (e.g., cell wall assembly, ribosome biogenesis, chromosome replication) that are of interest from the perspective of drug development (Babu et al., Mol Biosyst 12:1439-1455, 2009). We also show how genetic interactions generated by high-throughput eSGA screens can be validated by manual small-scale genetic crosses and by genetic complementation and gene rescue experiments.</p>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            
            
            <a href="/BabuLab/publication/#2">
              Journal article
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9"><em>Methods Mol. Biol.</em></div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style"></div>

    








<div class="share-box" aria-hidden="true">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=http://192.157.236.170/BabuLab/publication/pmid-21815091/&amp;text=Array-based%20synthetic%20genetic%20screens%20to%20map%20bacterial%20pathways%20and%20functional%20networks%20in%20Escherichia%20coli." target="_blank" rel="noopener" class="share-btn-twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=http://192.157.236.170/BabuLab/publication/pmid-21815091/&amp;t=Array-based%20synthetic%20genetic%20screens%20to%20map%20bacterial%20pathways%20and%20functional%20networks%20in%20Escherichia%20coli." target="_blank" rel="noopener" class="share-btn-facebook">
          <i class="fab fa-facebook"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Array-based%20synthetic%20genetic%20screens%20to%20map%20bacterial%20pathways%20and%20functional%20networks%20in%20Escherichia%20coli.&amp;body=http://192.157.236.170/BabuLab/publication/pmid-21815091/" target="_blank" rel="noopener" class="share-btn-email">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=http://192.157.236.170/BabuLab/publication/pmid-21815091/&amp;title=Array-based%20synthetic%20genetic%20screens%20to%20map%20bacterial%20pathways%20and%20functional%20networks%20in%20Escherichia%20coli." target="_blank" rel="noopener" class="share-btn-linkedin">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="whatsapp://send?text=Array-based%20synthetic%20genetic%20screens%20to%20map%20bacterial%20pathways%20and%20functional%20networks%20in%20Escherichia%20coli.%20http://192.157.236.170/BabuLab/publication/pmid-21815091/" target="_blank" rel="noopener" class="share-btn-whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=http://192.157.236.170/BabuLab/publication/pmid-21815091/&amp;title=Array-based%20synthetic%20genetic%20screens%20to%20map%20bacterial%20pathways%20and%20functional%20networks%20in%20Escherichia%20coli." target="_blank" rel="noopener" class="share-btn-weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>











  
  
  <div class="article-widget content-widget-hr">
    <h3>Related</h3>
    <ul>
      
      <li><a href="/BabuLab/publication/pmid-21877280/">Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli.</a></li>
      
      <li><a href="/BabuLab/publication/pmid-21219465/">A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair.</a></li>
      
      <li><a href="/BabuLab/publication/pmid-23168417/">Mapping bacterial functional networks and pathways in Escherichia Coli using synthetic genetic arrays.</a></li>
      
      <li><a href="/BabuLab/publication/pmid-24586182/">Quantitative genome-wide genetic interaction screens reveal global epistatic relationships of protein complexes in Escherichia coli.</a></li>
      
    </ul>
  </div>
  





  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js" integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js"></script>
        
      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js" integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin="anonymous"></script>
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/BabuLab/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/BabuLab/js/academic.min.66c553246b0f279a03be6e5597f72b52.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    <a href = "https://www.uregina.ca/contact/privacy-legal/">University of Regina Web Privacy and Legal Disclaimer</a>
  </p>
  <p class="powered-by">
    Â© 2020
  </p>

  
  






  <p class="powered-by">
    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>
</body>
</html>
