<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Dr. Mohan Babu">

  
  
  
    
  
  <meta name="description" content="Associate Professor">

  
  <link rel="alternate" hreflang="en-us" href="http://192.157.236.170/BabuLab/publication/">

  


  
  
  
  <meta name="theme-color" content="#085757">
  

  
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css" crossorigin="anonymous" title="hl-light" disabled>
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark">
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css" integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin="anonymous">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/BabuLab/css/academic.css">

  




  


  
  

  
  <link rel="alternate" href="/BabuLab/publication/index.xml" type="application/rss+xml" title="Babu Lab">
  

  <link rel="manifest" href="/BabuLab/index.webmanifest">
  <link rel="icon" type="image/png" href="/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="http://192.157.236.170/BabuLab/publication/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="Babu Lab">
  <meta property="og:url" content="http://192.157.236.170/BabuLab/publication/">
  <meta property="og:title" content="Publications | Babu Lab">
  <meta property="og:description" content="Associate Professor"><meta property="og:image" content="http://192.157.236.170/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="http://192.157.236.170/BabuLab/images/icon_hub5502365bf6553236465813059ac8b99_4054_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    <meta property="og:updated_time" content="2020-07-08T00:00:00&#43;00:00">
  

  




  


  





  <title>Publications | Babu Lab</title>

</head>
<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="dark">

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  









<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/BabuLab/">Babu Lab</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/BabuLab/">Babu Lab</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/labnews/"><span>News</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/labmember/"><span>Members</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link  active" href="/BabuLab/publication/"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/labresource/"><span>Resources</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/pgcf/"><span>PGCF</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/BabuLab/#contact"><span>Contact</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link js-theme-selector" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-palette" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>


  












  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1>Publications</h1>

  

  
</div>



<div class="universal-wrapper">
  <div class="row">
    <div class="col-lg-12">

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <div class="form-row mb-4">
        <div class="col-auto">
          <input type="search" class="filter-search" placeholder="Search..." autocapitalize="off"
          autocomplete="off" autocorrect="off" role="textbox" spellcheck="false">
        </div>
        <div class="col-auto">
          <select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group="pubtype">
            <option value="*">Type</option>
            
            
            <option value=".pubtype-2">
              Journal article
            </option>
            
          </select>
        </div>
        <div class="col-auto">
          <select class="pub-filters form-control form-control-sm" data-filter-group="year">
            <option value="*">Date</option>
            
            
            
            <option value=".year-2020">
              2020
            </option>
            
            <option value=".year-2019">
              2019
            </option>
            
            <option value=".year-2018">
              2018
            </option>
            
            <option value=".year-2017">
              2017
            </option>
            
            <option value=".year-2016">
              2016
            </option>
            
            <option value=".year-2015">
              2015
            </option>
            
            <option value=".year-2014">
              2014
            </option>
            
            <option value=".year-2013">
              2013
            </option>
            
            <option value=".year-2012">
              2012
            </option>
            
            <option value=".year-2011">
              2011
            </option>
            
            <option value=".year-2010">
              2010
            </option>
            
            <option value=".year-2009">
              2009
            </option>
            
            <option value=".year-2008">
              2008
            </option>
            
            <option value=".year-2005">
              2005
            </option>
            
            <option value=".year-2004">
              2004
            </option>
            
            
          </select>
        </div>
      </div>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Maryam Hajikarimlou</span>, <span >Houman Moteshareie</span>, <span >Katayoun Omidi</span>, <span >Mohsen Hooshyar</span>, <span >Sarah Shaikho</span>, <span >Tom Kazmirchuk</span>, <span >Daniel Burnside</span>, <span >Sarah Takallou</span>, <span >Narges Zare</span>, <span >Sasi Kumar Jagadeesan</span>, <span >Nathalie Puchacz</span>, <span >Mohan Babu</span>, <span >Myron Smith</span>, <span >Martin Holcik</span>, <span >Bahram Samanfar</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS ONE</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-32639961/">Sensitivity of yeast to lithium chloride connects the activity of YTA6 and YPR096C to translation of structured mRNAs.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-32639961/" class="summary-link">
    <div class="article-style">
      <p>Lithium Chloride (LiCl) toxicity, mode of action and cellular responses have been the subject of active investigations over the past decades. In yeast, LiCl treatment is reported to reduce the activity and alters the expression of PGM2, a gene that encodes a phosphoglucomutase involved in sugar metabolism. Reduced activity of phosphoglucomutase in the presence of galactose causes an accumulation of intermediate metabolites of galactose metabolism leading to a number of phenotypes including growth defect. In the current study, we identify two understudied yeast genes, YTA6 and YPR096C that when deleted, cell sensitivity to LiCl is increased when galactose is used as a carbon source. The 5&rsquo;-UTR of PGM2 mRNA is structured. Using this region, we show that YTA6 and YPR096C influence the translation of PGM2 mRNA.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-32639961/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0235033" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Vladimir Vélez-Segarra</span>, <span >Sahily González-Crespo</span>, <span >Ednalise Santiago-Cartagena</span>, <span >Luis E Vázquez-Quiñones</span>, <span >Nelson Martínez-Matías</span>, <span >Yamirelis Otero</span>, <span >Julián J Zayas</span>, <span >Rafael Siaca</span>, <span >Jeanmadi Del Rosario</span>, <span >Inoushka Mejías</span>, <span >José A Aponte</span>, <span >Noelani C Collazo</span>, <span >Francisco J Lasso</span>, <span >Jamie Snider</span>, <span >Matthew Jessulat</span>, <span >Hiroyuki Aoki</span>, <span >Brian C Rymond</span>, <span >Mohan Babu</span>, <span >Igor Stagljar</span>, <span >José R Rodríguez-Medina</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>G3 (Bethesda)</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-32641451/">Protein Interactions of the Mechanosensory Proteins Wsc2 and Wsc3 for Stress Resistance in Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-32641451/" class="summary-link">
    <div class="article-style">
      <p>Antifungal drug discovery and design is very challenging because of the considerable similarities in genetic features and metabolic pathways between fungi and humans. However, cell wall composition represents a notable point of divergence. Therefore, a research strategy was designed to improve our understanding of the mechanisms for maintaining fungal cell wall integrity, and to identify potential targets for new drugs that modulate the underlying protein-protein interactions in Saccharomyces cerevisiae This study defines roles for Wsc2p and Wsc3p and their interacting protein partners in the cell wall integrity signaling and cell survival mechanisms that respond to treatments with fluconazole and hydrogen peroxide. By combined genetic and biochemical approaches, we report the discovery of 12 novel protein interactors of Wsc2p and Wsc3p. Of these, Wsc2p interacting partners Gtt1p and Yck2p, have opposing roles in the resistance and sensitivity to fluconazole treatments respectively. The interaction of Wsc2p with Ras2p was confirmed by iMYTH and IP-MS approaches and is shown to play a dominant role in response to oxidative stress induced by hydrogen peroxide. Consistent with an earlier study, Ras2p was also identified as an interacting partner of Wsc1p and Mid2p cell wall integrity signaling proteins. Collectively, this study expands the interaction networks of the mechanosensory proteins of the Cell Wall Integrity pathway.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-32641451/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1534/g3.120.401468" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Siyi He</span>, <span >Mohamed Taha Moutaoufik</span>, <span >Saadul Islam</span>, <span >Amit Persad</span>, <span >Adam Wu</span>, <span >Khaled A Aly</span>, <span >Humphrey Fonge</span>, <span >Mohan Babu</span>, <span >Francisco S Cayabyab</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Biochim Biophys Acta Rev Cancer</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-32135169/">HERG channel and cancer: A mechanistic review of carcinogenic processes and therapeutic potential.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-32135169/" class="summary-link">
    <div class="article-style">
      <p>The human ether-à-go-go related gene (HERG) encodes the alpha subunit of Kv11.1, which is a voltage-gated K+ channel protein mainly expressed in heart and brain tissue. HERG plays critical role in cardiac repolarization, and mutations in HERG can cause long QT syndrome. More recently, evidence has emerged that HERG channels are aberrantly expressed in many kinds of cancer cells and play important roles in cancer progression. HERG could therefore be a potential biomarker for cancer and a possible molecular target for anticancer drug design. HERG affects a number of cellular processes, including cell proliferation, apoptosis, angiogenesis and migration, any of which could be affected by dysregulation of HERG. This review provides an overview of available information on HERG channel as it relates to cancer, with focus on the mechanism by which HERG influences cancer progression. Molecular docking attempts suggest two possible protein-protein interactions of HERG with the ß1-integrin receptor and the transcription factor STAT-1 as novel HERG-directed therapeutic targeting which avoids possible cardiotoxicity. The role of epigenetics in regulating HERG channel expression and activity in cancer will also be discussed. Finally, given its inherent extracellular accessibility as an ion channel, we discuss regulatory roles of this molecule in cancer physiology and therapeutic potential. Future research should be directed to explore the possibilities of therapeutic interventions targeting HERG channels while minding possible complications.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-32135169/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.bbcan.2020.188355" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Reza Pourhaghighi</span>, <span >Peter E A Ash</span>, <span >Sadhna Phanse</span>, <span >Florian Goebels</span>, <span >Lucas Z M Hu</span>, <span >Siwei Chen</span>, <span >Yingying Zhang</span>, <span >Shayne D Wierbowski</span>, <span >Samantha Boudeau</span>, <span >Mohamed T Moutaoufik</span>, <span >Ramy H Malty</span>, <span >Edyta Malolepsza</span>, <span >Kalliopi Tsafou</span>, <span >Aparna Nathan</span>, <span >Graham Cromar</span>, <span >Hongbo Guo</span>, <span >Ali Al Abdullatif</span>, <span >Daniel J Apicco</span>, <span >Lindsay A Becker</span>, <span >Aaron D Gitler</span>, <span >Stefan M Pulst</span>, <span >Ahmed Youssef</span>, <span >Ryan Hekman</span>, <span >Pierre C Havugimana</span>, <span >Carl A White</span>, <span >Benjamin C Blum</span>, <span >Antonia Ratti</span>, <span >Camron D Bryant</span>, <span >John Parkinson</span>, <span >Kasper Lage</span>, <span >Mohan Babu</span>, <span >Haiyuan Yu</span>, <span >Gary D Bader</span>, <span >Benjamin Wolozin</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Syst</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/BabuLab/publication/pmid-32325033/">
      <img src="/BabuLab/publication/pmid-32325033/featured_hu0f2ba2342e2cbc4713bdcc387ba3104b_5025767_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="BraInMap Elucidates the Macromolecular Connectivity Landscape of Mammalian Brain.">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-32325033/">BraInMap Elucidates the Macromolecular Connectivity Landscape of Mammalian Brain.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-32325033/" class="summary-link">
    <div class="article-style">
      <p>Connectivity webs mediate the unique biology of the mammalian brain. Yet, while cell circuit maps are increasingly available, knowledge of their underlying molecular networks remains limited. Here, we applied multi-dimensional biochemical fractionation with mass spectrometry and machine learning to survey endogenous macromolecules across the adult mouse brain. We defined a global &ldquo;interactome&rdquo; comprising over one thousand multi-protein complexes. These include hundreds of brain-selective assemblies that have distinct physical and functional attributes, show regional and cell-type specificity, and have links to core neurological processes and disorders. Using reciprocal pull-downs and a transgenic model, we validated a putative 28-member RNA-binding protein complex associated with amyotrophic lateral sclerosis, suggesting a coordinated function in alternative splicing in disease progression. This brain interaction map (BraInMap) resource facilitates mechanistic exploration of the unique molecular machinery driving core cellular processes of the central nervous system. It is publicly available and can be explored here <a href="https://www.bu.edu/dbin/cnsb/mousebrain/">https://www.bu.edu/dbin/cnsb/mousebrain/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-32325033/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.cels.2020.03.003" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mara Zilocchi</span>, <span >Mohamed Taha Moutaoufik</span>, <span >Matthew Jessulat</span>, <span >Sadhna Phanse</span>, <span >Khaled A Aly</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Expert Rev Proteomics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31986926/">Misconnecting the dots: altered mitochondrial protein-protein interactions and their role in neurodegenerative disorders.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31986926/" class="summary-link">
    <div class="article-style">
      <p>Introduction: Mitochondria (mt) are protein-protein interaction (PPI) hubs in the cell where mt-localized and associated proteins interact in a fashion critical for cell fitness. Altered mtPPIs are linked to neurodegenerative disorders (NDs) and drivers of pathological associations to mediate ND progression. Mapping altered mtPPIs will reveal how mt dysfunction is linked to NDs.Areas covered: This review discusses how database sources reflect on the number of mt protein or interaction predictions, and serves as an update on mtPPIs in mt dynamics and homeostasis. Emphasis is given to mRNA expression profiles for mt proteins in human tissues, cellular models relevant to NDs, and altered mtPPIs in NDs such as Parkinson&rsquo;s disease (PD), Amyotrophic lateral sclerosis (ALS) and Alzheimer&rsquo;s disease (AD).Expert opinion: We highlight the scarcity of biomarkers to improve diagnostic accuracy and tracking of ND progression, obstacles in recapitulating NDs using human cellular models to underpin the pathophysiological mechanisms of disease, and the shortage of mt protein interactome reference database(s) of neuronal cells. These bottlenecks are addressed by improvements in induced pluripotent stem cell creation and culturing, patient-derived 3D brain organoids to recapitulate structural arrangements of the brain, and cell sorting to elucidate mt proteome disparities between cell types.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31986926/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1080/14789450.2020.1723419" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ashwani Kumar</span>, <span >Ali Hosseinnia</span>, <span >Alla Gagarinova</span>, <span >Sadhna Phanse</span>, <span >Sunyoung Kim</span>, <span >Khaled A Aly</span>, <span >Sandra Zilles</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31504172/">A Gaussian process-based definition reveals new and bona fide genetic interactions compared to a multiplicative model in the Gram-negative Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31504172/" class="summary-link">
    <div class="article-style">
      <p>A digenic genetic interaction (GI) is observed when mutations in two genes within the same organism yield a phenotype that is different from the expected, given each mutation&rsquo;s individual effects. While multiplicative scoring is widely applied to define GIs, revealing underlying gene functions, it remains unclear if it is the most suitable choice for scoring GIs in Escherichia coli. Here, we assess many different definitions, including the multiplicative model, for mapping functional links between genes and pathways in E.coli.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31504172/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btz673" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mara Zilocchi</span>, <span >Ilaria Colugnat</span>, <span >Marta Lualdi</span>, <span >Monica Meduri</span>, <span >Federica Marini</span>, <span >Victor Corasolla Carregari</span>, <span >Mohamed Taha Moutaoufik</span>, <span >Sadhna Phanse</span>, <span >Luisa Pieroni</span>, <span >Mohan Babu</span>, <span >Barbara Garavaglia</span>, <span >Mauro Fasano</span>, <span >Tiziana Alberio</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Front Cell Dev Biol</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-32596240/">Exploring the Impact of PARK2 Mutations on the Total and Mitochondrial Proteome of Human Skin Fibroblasts.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-32596240/" class="summary-link">
    <div class="article-style">
      <p>Mutations in PARK2 gene are the most frequent cause of familial forms of Parkinson&rsquo;s disease (PD). This gene encodes Parkin, an E3 ubiquitin ligase involved in several cellular mechanisms, including mitophagy. Parkin loss-of-function is responsible for the cellular accumulation of damaged mitochondria, which in turn determines an increment of reactive oxygen species (ROS) levels, lower ATP production, and apoptosis activation. Given the importance of mitochondrial dysfunction and mitophagy impairment in PD pathogenesis, the aim of the present study was to investigate both total and mitochondrial proteome alterations in human skin fibroblasts of PARK2-mutated patients. To this end, both total and mitochondria-enriched protein fractions from fibroblasts of five PARK2-mutated patients and five control subjects were analyzed by quantitative shotgun proteomics to identify proteins specifically altered by Parkin mutations (mass spectrometry proteomics data have been submitted to ProteomeXchange with the identifier PXD015880). Both the network-based and gene set enrichment analyses pointed out pathways in which Rab GTPase proteins are involved. To have a more comprehensive view of the mitochondrial alterations due to PARK2 mutations, we investigated the impact of Parkin loss on mitochondrial function and network morphology. We unveiled that the mitochondrial membrane potential was reduced in PARK2-mutated patients, without inducing PINK1 accumulation, even when triggered with the ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP). Lastly, the analysis of the mitochondrial network morphology did not reveal any significant alterations in PARK2-mutated patients compared to control subjects. Thus, our results suggested that the network morphology was not influenced by the mitochondrial depolarization and by the lack of Parkin, revealing a possible impairment of fission and, more in general, of mitochondrial dynamics. In conclusion, the present work highlighted new molecular factors and pathways altered by PARK2 mutations, which will unravel possible biochemical pathways altered in the sporadic form of PD.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-32596240/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fcell.2020.00423" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Bálint Kintses</span>, <span >Pramod K Jangir</span>, <span >Gergely Fekete</span>, <span >Mónika Számel</span>, <span >Orsolya Méhi</span>, <span >Réka Spohn</span>, <span >Lejla Daruka</span>, <span >Ana Martins</span>, <span >Ali Hosseinnia</span>, <span >Alla Gagarinova</span>, <span >Sunyoung Kim</span>, <span >Sadhna Phanse</span>, <span >Bálint Csörgő</span>, <span >Ádám Györkei</span>, <span >Eszter Ari</span>, <span >Viktória Lázár</span>, <span >István Nagy</span>, <span >Mohan Babu</span>, <span >Csaba Pál</span>, <span >Balázs Papp</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat Commun</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31844052/">Chemical-genetic profiling reveals limited cross-resistance between antimicrobial peptides with different modes of action.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31844052/" class="summary-link">
    <div class="article-style">
      <p>Antimicrobial peptides (AMPs) are key effectors of the innate immune system and promising therapeutic agents. Yet, knowledge on how to design AMPs with minimal cross-resistance to human host-defense peptides remains limited. Here, we systematically assess the resistance determinants of Escherichia coli against 15 different AMPs using chemical-genetics and compare to the cross-resistance spectra of laboratory-evolved AMP-resistant strains. Although generalizations about AMP resistance are common in the literature, we find that AMPs with different physicochemical properties and cellular targets vary considerably in their resistance determinants. As a consequence, cross-resistance is prevalent only between AMPs with similar modes of action. Finally, our screen reveals several genes that shape susceptibility to membrane- and intracellular-targeting AMPs in an antagonistic manner. We anticipate that chemical-genetic approaches could inform future efforts to minimize cross-resistance between therapeutic and human host AMPs.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31844052/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-019-13618-z" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohamed Taha Moutaoufik</span>, <span >Ramy Malty</span>, <span >Shahreen Amin</span>, <span >Qingzhou Zhang</span>, <span >Sadhna Phanse</span>, <span >Alla Gagarinova</span>, <span >Mara Zilocchi</span>, <span >Larissa Hoell</span>, <span >Zoran Minic</span>, <span >Maria Gagarinova</span>, <span >Hiroyuki Aoki</span>, <span >Jocelyn Stockwell</span>, <span >Matthew Jessulat</span>, <span >Florian Goebels</span>, <span >Kirsten Broderick</span>, <span >Nichollas E Scott</span>, <span >James Vlasblom</span>, <span >Gabriel Musso</span>, <span >Bhanu Prasad</span>, <span >Eleonora Lamantea</span>, <span >Barbara Garavaglia</span>, <span >Alex Rajput</span>, <span >Kei Murayama</span>, <span >Yasushi Okazaki</span>, <span >Leonard J Foster</span>, <span >Gary D Bader</span>, <span >Francisco S Cayabyab</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>iScience</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31536960/">Rewiring of the Human Mitochondrial Interactome during Neuronal Reprogramming Reveals Regulators of the Respirasome and Neurogenesis.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31536960/" class="summary-link">
    <div class="article-style">
      <p>Mitochondrial protein (MP) assemblies undergo alterations during neurogenesis, a complex process vital in brain homeostasis and disease. Yet which MP assemblies remodel during differentiation remains unclear. Here, using mass spectrometry-based co-fractionation profiles and phosphoproteomics, we generated mitochondrial interaction maps of human pluripotent embryonal carcinoma stem cells and differentiated neuronal-like cells, which presented as two discrete cell populations by single-cell RNA sequencing. The resulting networks, encompassing 6,442 high-quality associations among 600 MPs, revealed widespread changes in mitochondrial interactions and site-specific phosphorylation during neuronal differentiation. By leveraging the networks, we show the orphan C20orf24 as a respirasome assembly factor whose disruption markedly reduces respiratory chain activity in patients deficient in complex IV. We also find that a heme-containing neurotrophic factor, neuron-derived neurotrophic factor [NENF], couples with Parkinson disease-related proteins to promote neurotrophic activity. Our results provide insights into the dynamic reorganization of mitochondrial networks during neuronal differentiation and highlights mechanisms for MPs in respirasome, neuronal function, and mitochondrial diseases.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31536960/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.isci.2019.08.057" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://tap.med.utoronto.ca/HsMTProt/php/home.php" target="_blank" rel="noopener">
    
    Database
  </a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.uregina.ca/external/communications/feature-stories/current/2019/10-01.html" target="_blank" rel="noopener">
    
    News
  </a>


  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Michael Luke Carlson</span>, <span >R Greg Stacey</span>, <span >John William Young</span>, <span >Irvinder Singh Wason</span>, <span >Zhiyu Zhao</span>, <span >David G Rattray</span>, <span >Nichollas Scott</span>, <span >Craig H Kerr</span>, <span >Mohan Babu</span>, <span >Leonard J Foster</span>, <span >Franck Duong Van Hoa</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Elife</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31364989/">Profiling the Escherichia coli membrane protein interactome captured in Peptidisc libraries.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31364989/" class="summary-link">
    <div class="article-style">
      <p>Protein-correlation-profiling (PCP), in combination with quantitative proteomics, has emerged as a high-throughput method for the rapid identification of dynamic protein complexes in native conditions. While PCP has been successfully applied to soluble proteomes, characterization of the membrane interactome has lagged, partly due to the necessary use of detergents to maintain protein solubility. Here, we apply the peptidisc, a &lsquo;one-size fits all&rsquo; membrane mimetic, for the capture of the Escherichia coli cell envelope proteome and its high-resolution fractionation in the absence of detergent. Analysis of the SILAC-labeled peptidisc library via PCP allows generation of over 4900 possible binary interactions out of &gt;700,000 random associations. Using well-characterized membrane protein systems such as the SecY translocon, the Bam complex and the MetNI transporter, we demonstrate that our dataset is a useful resource for identifying transient and surprisingly novel protein interactions. For example, we discover a trans-periplasmic supercomplex comprising subunits of the Bam and Sec machineries, including membrane-bound chaperones YfgM and PpiD. We identify RcsF and OmpA as bone fide interactors of BamA, and we show that MetQ association with the ABC transporter MetNI depends on its N-terminal lipid anchor. We also discover NlpA as a novel interactor of MetNI complex. Most of these interactions are largely undetected by standard detergent-based purification. Together, the peptidisc workflow applied to the proteomic field is emerging as a promising novel approach to characterize membrane protein interactions under native expression conditions and without genetic manipulation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31364989/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.7554/eLife.46615" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ednalise Santiago-Cartagena</span>, <span >Sahily González-Crespo</span>, <span >Vladimir Vélez</span>, <span >Nelson Martínez</span>, <span >Jamie Snider</span>, <span >Matthew Jessulat</span>, <span >Hiroyuki Aoki</span>, <span >Zoran Minic</span>, <span >Pearl Akamine</span>, <span >Inoushka Mejías</span>, <span >Luis M Pérez</span>, <span >Brian C Rymond</span>, <span >Mohan Babu</span>, <span >Igor Stagljar</span>, <span >José R Rodríguez-Medina</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>G3 (Bethesda)</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-30733383/">Identification and Functional Testing of Novel Interacting Protein Partners for the Stress Sensors Wsc1p and Mid2p of Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-30733383/" class="summary-link">
    <div class="article-style">
      <p>Wsc1p and Mid2p are transmembrane signaling proteins of cell wall stress in the budding yeast Saccharomyces cerevisiae When an environmental stress compromises cell wall integrity, they activate a cell response through the Cell Wall Integrity (CWI) pathway. Studies have shown that the cytoplasmic domain of Wsc1p initiates the CWI signaling cascade by interacting with Rom2p, a Rho1-GDP-GTP exchange factor. Binding of Rom2p to the cytoplasmic tail of Wsc1p requires dephosphorylation of specific serine residues but the mechanism by which the sensor is dephosphorylated and how it subsequently interacts with Rom2p remains unclear. We hypothesize that Wsc1p and Mid2p must be physically associated with interacting proteins other than Rom2p that facilitate its interaction and regulate the activation of CWI pathway. To address this, a cDNA plasmid library of yeast proteins was expressed in bait strains bearing membrane yeast two-hybrid (MYTH) reporter modules of Wsc1p and Mid2p, and their interacting preys were recovered and sequenced. 14 previously unreported interactors were confirmed for Wsc1p and 29 for Mid2p The interactors&rsquo; functionality were assessed by cell growth assays and CWI pathway activation by western blot analysis of Slt2p/Mpk1p phosphorylation in null mutants of each interactor under defined stress conditions. The susceptibility of these strains to different stresses were tested against antifungal agents and chemicals. This study reports important novel protein interactions of Wsc1p and Mid2p that are associated with the cellular response to oxidative stress induced by Hydrogen Peroxide and cell wall stress induced by Caspofungin.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-30733383/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1534/g3.118.200985" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Daniel Burnside</span>, <span >Andrew Schoenrock</span>, <span >Houman Moteshareie</span>, <span >Mohsen Hooshyar</span>, <span >Prabh Basra</span>, <span >Maryam Hajikarimlou</span>, <span >Kevin Dick</span>, <span >Brad Barnes</span>, <span >Tom Kazmirchuk</span>, <span >Matthew Jessulat</span>, <span >Sylvain Pitre</span>, <span >Bahram Samanfar</span>, <span >Mohan Babu</span>, <span >James R Green</span>, <span >Alex Wong</span>, <span >Frank Dehne</span>, <span >Kyle K Biggar</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>iScience</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-30660105/">In Silico Engineering of Synthetic Binding Proteins from Random Amino Acid Sequences.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-30660105/" class="summary-link">
    <div class="article-style">
      <p>Synthetic proteins with high affinity and selectivity for a protein target can be used as research tools, biomarkers, and pharmacological agents, but few methods exist to design such proteins de novo. To this end, the In-Silico Protein Synthesizer (InSiPS) was developed to design synthetic binding proteins (SBPs) that bind pre-determined targets while minimizing off-target interactions. InSiPS is a genetic algorithm that refines a pool of random sequences over hundreds of generations of mutation and selection to produce SBPs with pre-specified binding characteristics. As a proof of concept, we design SBPs against three yeast proteins and demonstrate binding and functional inhibition of two of three targets in vivo. Peptide SPOT arrays confirm binding sites, and a permutation array demonstrates target specificity. Our foundational approach will support the field of de novo design of small binding polypeptide motifs and has robust applicability while offering potential advantages over the limited number of techniques currently available.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-30660105/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.isci.2018.11.038" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mark F Mabanglo</span>, <span >Elisa Leung</span>, <span >Siavash Vahidi</span>, <span >Thiago V Seraphim</span>, <span >Bryan T Eger</span>, <span >Steve Bryson</span>, <span >Vaibhav Bhandari</span>, <span >Jin Lin Zhou</span>, <span >Yu-Qian Mao</span>, <span >Kamran Rizzolo</span>, <span >Marim M Barghash</span>, <span >Jordan D Goodreid</span>, <span >Sadhna Phanse</span>, <span >Mohan Babu</span>, <span >Leandro R S Barbosa</span>, <span >Carlos H I Ramos</span>, <span >Robert A Batey</span>, <span >Lewis E Kay</span>, <span >Emil F Pai</span>, <span >Walid A Houry</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Commun Biol</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31754640/">ClpP protease activation results from the reorganization of the electrostatic interaction networks at the entrance pores.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31754640/" class="summary-link">
    <div class="article-style">
      <p>Bacterial ClpP is a highly conserved, cylindrical, self-compartmentalizing serine protease required for maintaining cellular proteostasis. Small molecule acyldepsipeptides (ADEPs) and activators of self-compartmentalized proteases 1 (ACP1s) cause dysregulation and activation of ClpP, leading to bacterial cell death, highlighting their potential use as novel antibiotics. Structural changes in Neisseria meningitidis and Escherichia coli ClpP upon binding to novel ACP1 and ADEP analogs were probed by X-ray crystallography, methyl-TROSY NMR, and small angle X-ray scattering. ACP1 and ADEP induce distinct conformational changes in the ClpP structure. However, reorganization of electrostatic interaction networks at the ClpP entrance pores is necessary and sufficient for activation. Further activation is achieved by formation of ordered N-terminal axial loops and reduction in the structural heterogeneity of the ClpP cylinder. Activating mutations recapitulate the structural effects of small molecule activator binding. Our data, together with previous findings, provide a structural basis for a unified mechanism of compound-based ClpP activation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31754640/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s42003-019-0656-3" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Zhuoran Wu</span>, <span >Ramy Malty</span>, <span >Mohamed Taha Moutaoufik</span>, <span >Qingzhou Zhang</span>, <span >Matthew Jessulat</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Adv. Exp. Med. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-31452137/">A Tag-Based Affinity Purification Mass Spectrometry Workflow for Systematic Isolation of the Human Mitochondrial Protein Complexes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-31452137/" class="summary-link">
    <div class="article-style">
      <p>Mitochondria (mt) are double-membraned, dynamic organelles that play an essential role in a large number of cellular processes, and impairments in mt function have emerged as a causative factor for a growing number of human disorders. Given that most biological functions are driven by physical associations between proteins, the first step towards understanding mt dysfunction is to map its protein-protein interaction (PPI) network in a comprehensive and systematic fashion. While mass-spectrometry (MS) based approaches possess the high sensitivity ideal for such an endeavor, it also requires stringent biochemical purification of bait proteins to avoid detecting spurious, non-specific PPIs. Here, we outline a tagging-based affinity purification coupled with mass spectrometry (AP-MS) workflow for discovering new mt protein associations and providing novel insights into their role in mt biology and human physiology/pathology. Because AP-MS relies on the creation of proteins fused with affinity tags, we employ a versatile-affinity (VA) tag, consisting of 3× FLAG, 6 × His, and Strep III epitopes. For efficient delivery of affinity-tagged open reading frames (ORF) into mammalian cells, the VA-tag is cloned onto a specific ORF using Gateway recombinant cloning, and the resulting expression vector is stably introduced in target cells using lentiviral transduction. In this chapter, we show a functional workflow for mapping the mt interactome that includes tagging, stable transduction, selection and expansion of mammalian cell lines, mt extraction, identification of interacting protein partners by AP-MS, and lastly, computational assessment of protein complexes/PPI networks.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-31452137/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-981-13-8367-0_6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Zoran Minic</span>, <span >Tanya E S Dahms</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Chromatogr. B Analyt. Technol. Biomed. Life Sci.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-30380468/">Chromatographic separation strategies for precision mass spectrometry to study protein-protein interactions and protein phosphorylation.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-30380468/" class="summary-link">
    <div class="article-style">
      <p>Investigating protein-protein interactions and protein phosphorylation can be of great significance when studying biological processes and human diseases at the molecular level. However, sample complexity, presence of low abundance proteins, and dynamic nature of the proteins often impede in achieving sufficient analytical depth in proteomics research. In this regard, chromatographic separation methodologies have played a vital role in the identification and quantification of proteins in complex sample mixtures. The combination of peptide and protein fractionation techniques with advanced high-performance mass spectrometry has allowed the researchers to successfully study the protein-protein interactions and protein phosphorylation. Several new fractionation strategies for large scale analysis of proteins and peptides have been developed to study protein-protein interactions and protein phosphorylation. These emerging chromatography methodologies have enabled the identification of several hundred protein complexes and even thousands of phosphorylation sites in a single study. In this review, we focus on current workflow strategies and chromatographic tools, highlighting their advantages and disadvantages, and examining their associated challenges and future potential.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-30380468/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.jchromb.2018.10.022" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Irina A Rodionova</span>, <span >Norman Goodacre</span>, <span >Jimmy Do</span>, <span >Ali Hosseinnia</span>, <span >Mohan Babu</span>, <span >Peter Uetz</span>, <span >Milton H Saier</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Biol. Chem.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-30089654/">The uridylyltransferase GlnD and tRNA modification GTPase MnmE allosterically control Escherichia coli folylpoly-γ-glutamate synthase FolC.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-30089654/" class="summary-link">
    <div class="article-style">
      <p>Folate derivatives are important cofactors for enzymes in several metabolic processes. Folate-related inhibition and resistance mechanisms in bacteria are potential targets for antimicrobial therapies and therefore a significant focus of current research. Here, we report that the activity of Escherichia coli poly-γ-glutamyl tetrahydrofolate/dihydrofolate synthase (FolC) is regulated by glutamate/glutamine-sensing uridylyltransferase (GlnD), THF-dependent tRNA modification enzyme (MnmE), and UDP-glucose dehydrogenase (Ugd) as shown by direct in vitro protein-protein interactions. Using kinetics analyses, we observed that GlnD, Ugd, and MnmE activate FolC many-fold by decreasing the Khalf of FolC for its substrate l-glutamate. Moreover, FolC inhibited the GTPase activity of MnmE at low GTP concentrations. The growth phenotypes associated with these proteins are discussed. These results, obtained using direct in vitro enzyme assays, reveal unanticipated networks of allosteric regulatory interactions in the folate pathway in E. coli and indicate regulation of polyglutamylated tetrahydrofolate biosynthesis by the availability of nitrogen sources, signaled by the glutamine-sensing GlnD protein.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-30089654/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1074/jbc.RA118.004425" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Keith S Wong</span>, <span >Mark F Mabanglo</span>, <span >Thiago V Seraphim</span>, <span >Antonio Mollica</span>, <span >Yu-Qian Mao</span>, <span >Kamran Rizzolo</span>, <span >Elisa Leung</span>, <span >Mohamed T Moutaoufik</span>, <span >Larissa Hoell</span>, <span >Sadhna Phanse</span>, <span >Jordan Goodreid</span>, <span >Leandro R S Barbosa</span>, <span >Carlos H I Ramos</span>, <span >Mohan Babu</span>, <span >Vito Mennella</span>, <span >Robert A Batey</span>, <span >Aaron D Schimmer</span>, <span >Walid A Houry</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Chem Biol</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-30126533/">Acyldepsipeptide Analogs Dysregulate Human Mitochondrial ClpP Protease Activity and Cause Apoptotic Cell Death.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-30126533/" class="summary-link">
    <div class="article-style">
      <p>Acyldepsipeptides (ADEPs) are potential antibiotics that dysregulate the activity of the highly conserved tetradecameric bacterial ClpP protease, leading to bacterial cell death. Here, we identified ADEP analogs that are potent dysregulators of the human mitochondrial ClpP (HsClpP). These ADEPs interact tightly with HsClpP, causing the protease to non-specifically degrade model substrates. Dysregulation of HsClpP activity by ADEP was found to induce cytotoxic effects via activation of the intrinsic, caspase-dependent apoptosis. ADEP-HsClpP co-crystal structure was solved for one of the analogs revealing a highly complementary binding interface formed by two HsClpP neighboring subunits but, unexpectedly, with HsClpP in the compact conformation. Given that HsClpP is highly expressed in multiple cancers and has important roles in cell metastasis, our findings suggest a therapeutic potential for ADEPs in cancer treatment.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-30126533/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.chembiol.2018.05.014" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Behzad M Toosi</span>, <span >Amr El Zawily</span>, <span >Luke Truitt</span>, <span >Matthew Shannon</span>, <span >Odette Allonby</span>, <span >Mohan Babu</span>, <span >John DeCoteau</span>, <span >Darrell Mousseau</span>, <span >Mohsin Ali</span>, <span >Tanya Freywald</span>, <span >Amanda Gall</span>, <span >Frederick S Vizeacoumar</span>, <span >Morgan W Kirzinger</span>, <span >C Ronald Geyer</span>, <span >Deborah H Anderson</span>, <span >TaeHyung Kim</span>, <span >Alana L Welm</span>, <span >Peter Siegel</span>, <span >Franco J Vizeacoumar</span>, <span >Anthony Kusalik</span>, <span >Andrew Freywald</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Oncogene</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29700392/">EPHB6 augments both development and drug sensitivity of triple-negative breast cancer tumours.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29700392/" class="summary-link">
    <div class="article-style">
      <p>Triple-negative breast cancer (TNBC) tumours that lack expression of oestrogen, and progesterone receptors, and do not overexpress the HER2 receptor represent the most aggressive breast cancer subtype, which is characterised by the resistance to therapy in frequently relapsing tumours and a high rate of patient mortality. This is likely due to the resistance of slowly proliferating tumour-initiating cells (TICs), and understanding molecular mechanisms that control TICs behaviour is crucial for the development of effective therapeutic approaches. Here, we present our novel findings, indicating that an intrinsically catalytically inactive member of the Eph group of receptor tyrosine kinases, EPHB6, partially suppresses the epithelial-mesenchymal transition in TNBC cells, while also promoting expansion of TICs. Our work reveals that EPHB6 interacts with the GRB2 adapter protein and that its effect on enhancing cell proliferation is mediated by the activation of the RAS-ERK pathway, which allows it to elevate the expression of the TIC-related transcription factor, OCT4. Consistent with this, suppression of either ERK or OCT4 activities blocks EPHB6-induced pro-proliferative responses. In line with its ability to trigger propagation of TICs, EPHB6 accelerates tumour growth, potentiates tumour initiation and increases TIC populations in xenograft models of TNBC. Remarkably, EPHB6 also suppresses tumour drug resistance to DNA-damaging therapy, probably by forcing TICs into a more proliferative, drug-sensitive state. In agreement, patients with higher EPHB6 expression in their tumours have a better chance for recurrence-free survival. These observations describe an entirely new mechanism that governs TNBC and suggest that it may be beneficial to enhance EPHB6 action concurrent with applying a conventional DNA-damaging treatment, as it would decrease drug resistance and improve tumour elimination.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29700392/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41388-018-0228-x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Kamran Rizzolo</span>, <span >Ashwani Kumar</span>, <span >Yoshito Kakihara</span>, <span >Sadhna Phanse</span>, <span >Zoran Minic</span>, <span >Jamie Snider</span>, <span >Igor Stagljar</span>, <span >Sandra Zilles</span>, <span >Mohan Babu</span>, <span >Walid A Houry</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol Omics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29659649/">Systems analysis of the genetic interaction network of yeast molecular chaperones.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29659649/" class="summary-link">
    <div class="article-style">
      <p>Molecular chaperones are typically promiscuous interacting proteins that function globally in the cell to maintain protein homeostasis. Recently, we had carried out experiments that elucidated a comprehensive interaction network for the core 67 chaperones and 15 cochaperones in the budding yeast Saccharomyces cerevisiae [Rizzolo et al., Cell Rep., 2017, 20, 2735-2748]. Here, the genetic (i.e. epistatic) interaction network obtained for chaperones was further analyzed, revealing that the global topological parameters of the resulting network have a more central role in mediating interactions in comparison to the rest of the proteins in the cell. Most notably, we observed Hsp10, Hsp70 Ssz1 chaperone, and Hsp90 cochaperone Cdc37 to be the main drivers of the network architecture. Systematic analysis on the physicochemical properties for all chaperone interactors further revealed the presence of preferential domains and folds that are highly interactive with chaperones such as the WD40 repeat domain. Further analysis with established cellular complexes revealed the involvement of R2TP chaperone in quaternary structure formation. Our results thus provide a global overview of the chaperone network properties in yeast, expanding our understanding of their functional diversity and their role in protein homeostasis.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29659649/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1039/C7MO00142H" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Irina A Rodionova</span>, <span >Norman Goodacre</span>, <span >Mohan Babu</span>, <span >Andrew Emili</span>, <span >Peter Uetz</span>, <span >Milton H Saier</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Bacteriol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29229699/">The Nitrogen Regulatory PII Protein (GlnB) and N-Acetylglucosamine 6-Phosphate Epimerase (NanE) Allosterically Activate Glucosamine 6-Phosphate Deaminase (NagB) in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29229699/" class="summary-link">
    <div class="article-style">
      <p>Amino sugars are good sources of both ammonia and fructose-6-phosphate, produced by the glucosamine 6-phosphate deaminase, NagB. NagB is known to be allosterically regulated by N-acetylglucosamine 6-phosphate (GlcNAc-6P) and the phosphocarrier protein of the bacterial phosphotransferase system, HPr, in Escherichia coli We provide evidence that NanE, GlcNAc-6P epimerase, and the uridylylated PII protein (U-PII) also allosterically activate NagB by direct protein-protein interactions. NanE is essential for neuraminic acid (NANA) and N-acetylmannosamine (ManNAc) utilization, and PII is known to be a central metabolic nitrogen regulator. We demonstrate that uridylylated PII (but not underivatized PII) activates NagB &gt;10-fold at low concentrations of substrate, whereas NanE increases NagB activity &gt;2-fold. NanE activates NagB in the absence or presence of GlcNAc-6P, but HPr and U-PII activation requires the presence of GlcNAc-6P. Activation of NagB by HPr and uridylylated PII, as well as by NanE and HPr (but not by NanE and U-PII), is synergistic, and the modeling, which suggests specific residues involved in complex formation, provides possible explanations. Specific physiological functions for the regulation of NagB by its three protein activators are proposed. Each regulatory agent is suggested to mediate signal transduction in response to a different stimulus.IMPORTANCE The regulation of amino sugar utilization is important for the survival of bacteria in a competitive environment. NagB, a glucosamine 6-phosphate deaminase in Escherichia coli, is essential for amino sugar utilization and is known to be allosterically regulated by N-acetylglucosamine 6-phosphate (GlcNAc-6P) and the histidine-phosphorylatable phosphocarrier protein, HPr. We provide evidence here that NanE, GlcNAc-6P epimerase, and the uridylylated PII protein allosterically activate NagB by direct protein-protein interactions. NanE is essential for N-acetylneuraminic acid (NANA) and N-acetylmannosamine (ManNAc) utilization, and the PII protein is known to be a central metabolic nitrogen regulator. Regulatory links between carbon and nitrogen metabolism are important for adaptation of metabolism to different growth conditions.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29229699/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/JB.00691-17" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Imelda Galván Márquez</span>, <span >Mergan Ghiyasvand</span>, <span >Andrey Massarsky</span>, <span >Mohan Babu</span>, <span >Bahram Samanfar</span>, <span >Katayoun Omidi</span>, <span >Thomas W Moon</span>, <span >Myron L Smith</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS ONE</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29554091/">Zinc oxide and silver nanoparticles toxicity in the baker&#39;s yeast, Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29554091/" class="summary-link">
    <div class="article-style">
      <p>Engineered nanomaterials (ENMs) are increasingly incorporated into a variety of commercial applications and consumer products; however, ENMs may possess cytotoxic properties due to their small size. This study assessed the effects of two commonly used ENMs, zinc oxide nanoparticles (ZnONPs) and silver nanoparticles (AgNPs), in the model eukaryote Saccharomyces cerevisiae. A collection of ≈4600 S. cerevisiae deletion mutant strains was used to deduce the genes, whose absence makes S. cerevisiae more prone to the cytotoxic effects of ZnONPs or AgNPs. We demonstrate that S. cerevisiae strains that lack genes involved in transmembrane and membrane transport, cellular ion homeostasis, and cell wall organization or biogenesis exhibited the highest sensitivity to ZnONPs. In contrast, strains that lack genes involved in transcription and RNA processing, cellular respiration, and endocytosis and vesicular transport exhibited the highest sensitivity to AgNPs. Secondary assays confirmed that ZnONPs affected cell wall function and integrity, whereas AgNPs exposure decreased transcription, reduced endocytosis, and led to a dysfunctional electron transport system. This study supports the use of S. cerevisiae Gene Deletion Array as an effective high-throughput technique to determine cellular targets of ENM toxicity.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29554091/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0193111" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Katayoun Omidi</span>, <span >Matthew Jessulat</span>, <span >Mohsen Hooshyar</span>, <span >Daniel Burnside</span>, <span >Andrew Schoenrock</span>, <span >Tom Kazmirchuk</span>, <span >Maryam Hajikarimlou</span>, <span >Mary Daniel</span>, <span >Houman Moteshareie</span>, <span >Urvi Bhojoo</span>, <span >Megan Sanders</span>, <span >Dindial Ramotar</span>, <span >Frank Dehne</span>, <span >Bahram Samanfar</span>, <span >Mohan Babu</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Gene</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28987344/">Uncharacterized ORF HUR1 influences the efficiency of non-homologous end-joining repair in Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28987344/" class="summary-link">
    <div class="article-style">
      <p>Non-Homologous End Joining (NHEJ) is a highly conserved pathway that repairs Double-Strand Breaks (DSBs) within DNA. Here we show that the deletion of yeast uncharacterized ORF HUR1, Hydroxyurea Resistance1 affects the efficiency of NHEJ. Our findings are supported by Protein-Protein Interaction (PPI), genetic interaction and drug sensitivity analyses. To assess the activity of HUR1 in DSB repair, we deleted its non-overlapping region with PMR1, referred to as HUR1-A. We observed that similar to deletion of TPK1 and NEJ1, and unlike YKU70 (important for NHEJ of DNA with overhang and not blunt end), deletion of HUR1-A reduced the efficiency of NHEJ in both overhang and blunt end plasmid repair assays. Similarly, a chromosomal repair assay showed a reduction for repair efficiency when HUR1-A was deleted. In agreement with a functional connection for Hur1p with Tpk1p and NEJ1p, double mutant strains Δhur1-A/Δtpk1, and Δhur1-A/Δnej1 showed the same reduction in the efficiency of plasmid repair, compared to both single deletion strains. Also, using a Homologous Recombination (HR) specific plasmid-based DSB repair assay we observed that deletion of HUR1-A influenced the efficiency of HR repair, suggesting that HUR1 might also play additional roles in other DNA repair pathways.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28987344/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.gene.2017.10.003" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Houman Moteshareie</span>, <span >Maryam Hajikarimlou</span>, <span >Alex Mulet Indrayanti</span>, <span >Daniel Burnside</span>, <span >Ana Paula Dias</span>, <span >Clara Lettl</span>, <span >Duale Ahmed</span>, <span >Katayoun Omidi</span>, <span >Tom Kazmirchuk</span>, <span >Nathalie Puchacz</span>, <span >Narges Zare</span>, <span >Sarah Takallou</span>, <span >Thet Naing</span>, <span >Raúl Bonne Hernández</span>, <span >William G Willmore</span>, <span >Mohan Babu</span>, <span >Bruce McKay</span>, <span >Bahram Samanfar</span>, <span >Martin Holcik</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS ONE</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-30231023/">Heavy metal sensitivities of gene deletion strains for ITT1 and RPS1A connect their activities to the expression of URE2, a key gene involved in metal detoxification in yeast.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-30231023/" class="summary-link">
    <div class="article-style">
      <p>Heavy metal and metalloid contaminations are among the most concerning types of pollutant in the environment. Consequently, it is important to investigate the molecular mechanisms of cellular responses and detoxification pathways for these compounds in living organisms. To date, a number of genes have been linked to the detoxification process. The expression of these genes can be controlled at both transcriptional and translational levels. In baker&rsquo;s yeast, Saccharomyces cerevisiae, resistance to a wide range of toxic metals is regulated by glutathione S-transferases. Yeast URE2 encodes for a protein that has glutathione peroxidase activity and is homologous to mammalian glutathione S-transferases. The URE2 expression is critical to cell survival under heavy metal stress. Here, we report on the finding of two genes, ITT1, an inhibitor of translation termination, and RPS1A, a small ribosomal protein, that when deleted yeast cells exhibit similar metal sensitivity phenotypes to gene deletion strain for URE2. Neither of these genes were previously linked to metal toxicity. Our gene expression analysis illustrates that these two genes affect URE2 mRNA expression at the level of translation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-30231023/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0198704" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Cedoljub Bundalovic-Torma</span>, <span >Charles Calmettes</span>, <span >Sadhna Phanse</span>, <span >Qingzhou Zhang</span>, <span >Yue Jiang</span>, <span >Zoran Minic</span>, <span >Sunyoung Kim</span>, <span >Jitender Mehla</span>, <span >Alla Gagarinova</span>, <span >Irina Rodionova</span>, <span >Ashwani Kumar</span>, <span >Hongbo Guo</span>, <span >Olga Kagan</span>, <span >Oxana Pogoutse</span>, <span >Hiroyuki Aoki</span>, <span >Viktor Deineko</span>, <span >J Harry Caufield</span>, <span >Erik Holtzapple</span>, <span >Zhongge Zhang</span>, <span >Ake Vastermark</span>, <span >Yogee Pandya</span>, <span >Christine Chieh-Lin Lai</span>, <span >Majida El Bakkouri</span>, <span >Yogesh Hooda</span>, <span >Megha Shah</span>, <span >Dan Burnside</span>, <span >Mohsen Hooshyar</span>, <span >James Vlasblom</span>, <span >Sessandra V Rajagopala</span>, <span >Ashkan Golshani</span>, <span >Stefan Wuchty</span>, <span >Jack F Greenblatt</span>, <span >Milton Saier</span>, <span >Peter Uetz</span>, <span >Trevor F Moraes</span>, <span >John Parkinson</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat. Biotechnol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29176613/">Global landscape of cell envelope protein complexes in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29176613/" class="summary-link">
    <div class="article-style">
      <p>Bacterial cell envelope protein (CEP) complexes mediate a range of processes, including membrane assembly, antibiotic resistance and metabolic coordination. However, only limited characterization of relevant macromolecules has been reported to date. Here we present a proteomic survey of 1,347 CEPs encompassing 90% inner- and outer-membrane and periplasmic proteins of Escherichia coli. After extraction with non-denaturing detergents, we affinity-purified 785 endogenously tagged CEPs and identified stably associated polypeptides by precision mass spectrometry. The resulting high-quality physical interaction network, comprising 77% of targeted CEPs, revealed many previously uncharacterized heteromeric complexes. We found that the secretion of autotransporters requires translocation and the assembly module TamB to nucleate proper folding from periplasm to cell surface through a cooperative mechanism involving the β-barrel assembly machinery. We also establish that an ABC transporter of unknown function, YadH, together with the Mla system preserves outer membrane lipid asymmetry. This E. coli CEP &lsquo;interactome&rsquo; provides insights into the functional landscape governing CE systems essential to bacterial growth, metabolism and drug resistance.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29176613/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nbt.4024" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ashwani Kumar</span>, <span >Kamran Rizzolo</span>, <span >Sandra Zilles</span>, <span >Mohan Babu</span>, <span >Walid A Houry</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Methods Mol. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29177666/">Computational Analysis of the Chaperone Interaction Networks.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29177666/" class="summary-link">
    <div class="article-style">
      <p>We provide computational protocols to identify chaperone interacting proteins using a combination of both physical (protein-protein) and genetic (gene-gene or epistatic) interaction data derived from the published large-scale proteomic and genomic studies for the budding yeast Saccharomyces cerevisiae. Using these datasets, we discuss bioinformatic analyses that can be employed to build comprehensive high-fidelity chaperone interaction networks. Given that many proteins typically function as complexes in the cell, we highlight various step-wise approaches for combining both the genetic and physical interaction datasets to decipher intra- and inter-connections for distinct chaperone- and non-chaperone-containing complexes in the network. Together, these informatics procedures will aid in identifying protein complexes with distinctive functional specializations in the cell that yield a very broad and diverse set of interactions. The described procedures can also be leveraged to datasets from other eukaryotes, including humans.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29177666/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-1-4939-7477-1_20" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Gerrit Kürschner</span>, <span >Qingzhou Zhang</span>, <span >Rosanna Clima</span>, <span >Yi Xiao</span>, <span >Jonas Felix Busch</span>, <span >Ergin Kilic</span>, <span >Klaus Jung</span>, <span >Nikolaus Berndt</span>, <span >Sascha Bulik</span>, <span >Hermann-Georg Holzhütter</span>, <span >Giuseppe Gasparre</span>, <span >Marcella Attimonelli</span>, <span >Mohan Babu</span>, <span >David Meierhofer</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Oncotarget</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29285300/">Renal oncocytoma characterized by the defective complex I of the respiratory chain boosts the synthesis of the ROS scavenger glutathione.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29285300/" class="summary-link">
    <div class="article-style">
      <p>Renal oncocytomas are rare benign tumors of the kidney and characterized by a deficient complex I (CI) enzyme activity of the oxidative phosphorylation (OXPHOS) system caused by mitochondrial DNA (mtDNA) mutations. Yet, little is known about the underlying molecular mechanisms and alterations of metabolic pathways in this tumor. We compared renal oncocytomas with adjacent matched normal kidney tissues on a global scale by multi-omics approaches, including whole exome sequencing (WES), proteomics, metabolomics, and metabolic pathway simulation. The abundance of proteins localized to mitochondria increased more than 2-fold, the only exception was a strong decrease in the abundance for CI subunits that revealed several pathogenic heteroplasmic mtDNA mutations by WES. We also observed renal oncocytomas to dysregulate main metabolic pathways, shunting away from gluconeogenesis and lipid metabolism. Nevertheless, the abundance of energy carrier molecules such as NAD+, NADH, NADP, ATP, and ADP were significantly higher in renal oncocytomas. Finally, a substantial 5000-fold increase of the reactive oxygen species scavenger glutathione can be regarded as a new hallmark of renal oncocytoma. Our findings demonstrate that renal oncocytomas undergo a metabolic switch to eliminate ATP consuming processes to ensure a sufficient energy supply for the tumor.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29285300/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.18632/oncotarget.22413" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Tom Kazmirchuk</span>, <span >Kevin Dick</span>, <span >Daniel J Burnside</span>, <span >Brad Barnes</span>, <span >Houman Moteshareie</span>, <span >Maryam Hajikarimlou</span>, <span >Katayoun Omidi</span>, <span >Duale Ahmed</span>, <span >Andrew Low</span>, <span >Clara Lettl</span>, <span >Mohsen Hooshyar</span>, <span >Andrew Schoenrock</span>, <span >Sylvain Pitre</span>, <span >Mohan Babu</span>, <span >Edana Cassol</span>, <span >Bahram Samanfar</span>, <span >Alex Wong</span>, <span >Frank Dehne</span>, <span >James R Green</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Comput Biol Chem</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29112936/">Designing anti-Zika virus peptides derived from predicted human-Zika virus protein-protein interactions.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29112936/" class="summary-link">
    <div class="article-style">
      <p>The production of anti-Zika virus (ZIKV) therapeutics has become increasingly important as the propagation of the devastating virus continues largely unchecked. Notably, a causal relationship between ZIKV infection and neurodevelopmental abnormalities has been widely reported, yet a specific mechanism underlying impaired neurological development has not been identified. Here, we report on the design of several synthetic competitive inhibitory peptides against key pathogenic ZIKV proteins through the prediction of protein-protein interactions (PPIs). Often, PPIs between host and viral proteins are crucial for infection and pathogenesis, making them attractive targets for therapeutics. Using two complementary sequence-based PPI prediction tools, we first produced a comprehensive map of predicted human-ZIKV PPIs (involving 209 human protein candidates). We then designed several peptides intended to disrupt the corresponding host-pathogen interactions thereby acting as anti-ZIKV therapeutics. The data generated in this study constitute a foundational resource to aid in the multi-disciplinary effort to combat ZIKV infection, including the design of additional synthetic proteins.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29112936/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.compbiolchem.2017.10.011" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ramy H Malty</span>, <span >Hiroyuki Aoki</span>, <span >Ashwani Kumar</span>, <span >Sadhna Phanse</span>, <span >Shahreen Amin</span>, <span >Qingzhou Zhang</span>, <span >Zoran Minic</span>, <span >Florian Goebels</span>, <span >Gabriel Musso</span>, <span >Zhuoran Wu</span>, <span >Hosam Abou-Tok</span>, <span >Michael Meyer</span>, <span >Viktor Deineko</span>, <span >Sandy Kassir</span>, <span >Vishaldeep Sidhu</span>, <span >Matthew Jessulat</span>, <span >Nichollas E Scott</span>, <span >Xuejian Xiong</span>, <span >James Vlasblom</span>, <span >Bhanu Prasad</span>, <span >Leonard J Foster</span>, <span >Tiziana Alberio</span>, <span >Barbara Garavaglia</span>, <span >Haiyuan Yu</span>, <span >Gary D Bader</span>, <span >Ken Nakamura</span>, <span >John Parkinson</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Syst</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29128334/">A Map of Human Mitochondrial Protein Interactions Linked to Neurodegeneration Reveals New Mechanisms of Redox Homeostasis and NF-κB Signaling.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29128334/" class="summary-link">
    <div class="article-style">
      <p>Mitochondrial protein (MP) dysfunction has been linked to neurodegenerative disorders (NDs); however, the discovery of the molecular mechanisms underlying NDs has been impeded by the limited characterization of interactions governing MP function. Here, using mass spectrometry (MS)-based analysis of 210 affinity-purified mitochondrial (mt) fractions isolated from 27 epitope-tagged human ND-linked MPs in HEK293 cells, we report a high-confidence MP network including 1,964 interactions among 772 proteins (&gt;90% previously unreported). Nearly three-fourths of these interactions were confirmed in mouse brain and multiple human differentiated neuronal cell lines by primary antibody immunoprecipitation and MS, with many linked to NDs and autism. We show that the SOD1-PRDX5 interaction, critical for mt redox homeostasis, can be perturbed by amyotrophic lateral sclerosis-linked SOD1 allelic variants and establish a functional role for ND-linked factors coupled with IκBɛ in NF-κB activation. Our results identify mechanisms for ND-linked MPs and expand the human mt interaction landscape.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29128334/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.cels.2017.10.010" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Alla Gagarinova</span>, <span >Sadhna Phanse</span>, <span >Miroslaw Cygler</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Expert Rev Proteomics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28786313/">Insights from protein-protein interaction studies on bacterial pathogenesis.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28786313/" class="summary-link">
    <div class="article-style">
      <p>The threat bacterial pathogens pose to human health is increasing with the number and distribution of antibiotic-resistant bacteria, while the rate of discovery of new antimicrobials dwindles. Proteomics is playing key roles in understanding the molecular mechanisms of bacterial pathogenesis, and in identifying disease outcome determinants. The physical associations identified by proteomics can provide the means to develop pathogen-specific treatment methods that reduce the spread of antibiotic resistance and alleviate the negative effects of broad-spectrum antibiotics on beneficial bacteria. Areas covered: This review discusses recent trends in proteomics and introduces new and developing approaches that can be applied to the study of protein-protein interactions (PPIs) underlying bacterial pathogenesis. The approaches examined encompass options for mapping proteomes as well as stable and transient interactions in vivo and in vitro. We also explored the coverage of bacterial and human-bacterial PPIs, knowledge gaps in this area, and how they can be filled. Expert commentary: Identifying potential antimicrobial candidates is confounded by the complex molecular biology of bacterial pathogenesis and the lack of knowledge about PPIs underlying this process. Proteomics approaches can offer new perspectives for mechanistic insights and identify essential targets for guiding the discovery of next generation antimicrobials.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28786313/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1080/14789450.2017.1365603" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Sheila N López</span>, <span >Madeline Rodríguez-Valentín</span>, <span >Mariela Rivera</span>, <span >Maridaliz Rodríguez</span>, <span >Mohan Babu</span>, <span >Luis A Cubano</span>, <span >Huangui Xiong</span>, <span >Guangdi Wang</span>, <span >Lilia Kucheryavykh</span>, <span >Nawal M Boukli</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Oncotarget</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28978127/">HIV-1 Gp120 clade B/C induces a GRP78 driven cytoprotective mechanism in astrocytoma.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28978127/" class="summary-link">
    <div class="article-style">
      <p>HIV-1 clades are known to be one of the key factors implicated in modulating HIV-associated neurocognitive disorders. HIV-1 B and C clades account for the majority of HIV-1 infections, clade B being the most neuropathogenic. The mechanisms behind HIV-mediated neuropathogenesis remain the subject of active research. We hypothesized that HIV-1 gp120 clade B and C proteins may exert differential proliferation, cell survival and NeuroAIDS effects in human astrocytoma cells via the Unfolded Protein Response, an endoplasmic reticulum- based cytoprotective mechanism. The differential effect of gp120 clade B and C was evaluated using for the first time a Tandem Mass Tag isobaric labeling quantitative proteomic approach. Flow cytometry analyses were performed for cell cycle and cell death identification. Among the proteins differentiated by HIV-1 gp120 proteins figure cytoskeleton, oxidative stress, UPR markers and numerous glycolytic metabolism enzymes. Our results demonstrate that HIV-1 gp120 B induced migration, proliferative and protective responses granted by the expression of GRP78, while HIV-1 gp120 C induced the expression of key inflammatory and pro-apoptotic markers. These novel findings put forward the first evidence that GRP78 is a key player in HIV-1 clade B and C neuropathogenic discrepancies and can be used as a novel target for immunotherapies.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28978127/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.18632/oncotarget.19474" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Kamran Rizzolo</span>, <span >Jennifer Huen</span>, <span >Ashwani Kumar</span>, <span >Sadhna Phanse</span>, <span >James Vlasblom</span>, <span >Yoshito Kakihara</span>, <span >Hussein A Zeineddine</span>, <span >Zoran Minic</span>, <span >Jamie Snider</span>, <span >Wen Wang</span>, <span >Carles Pons</span>, <span >Thiago V Seraphim</span>, <span >Edgar Erik Boczek</span>, <span >Simon Alberti</span>, <span >Michael Costanzo</span>, <span >Chad L Myers</span>, <span >Igor Stagljar</span>, <span >Charles Boone</span>, <span >Mohan Babu</span>, <span >Walid A Houry</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Rep</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28903051/">Features of the Chaperone Cellular Network Revealed through Systematic Interaction Mapping.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28903051/" class="summary-link">
    <div class="article-style">
      <p>A comprehensive view of molecular chaperone function in the cell was obtained through a systematic global integrative network approach based on physical (protein-protein) and genetic (gene-gene or epistatic) interaction mapping. This allowed us to decipher interactions involving all core chaperones (67) and cochaperones (15) of Saccharomyces cerevisiae. Our analysis revealed the presence of a large chaperone functional supercomplex, which we named the naturally joined (NAJ) chaperone complex, encompassing Hsp40, Hsp70, Hsp90, AAA+, CCT, and small Hsps. We further found that many chaperones interact with proteins that form foci or condensates under stress conditions. Using an in vitro reconstitution approach, we demonstrate condensate formation for the highly conserved AAA+ ATPases Rvb1 and Rvb2, which are part of the R2TP complex that interacts with Hsp90. This expanded view of the chaperone network in the cell clearly demonstrates the distinction between chaperones having broad versus narrow substrate specificities in protein homeostasis.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28903051/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.celrep.2017.08.074" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Irina A Rodionova</span>, <span >Zhongge Zhang</span>, <span >Jitender Mehla</span>, <span >Norman Goodacre</span>, <span >Mohan Babu</span>, <span >Andrew Emili</span>, <span >Peter Uetz</span>, <span >Milton H Saier</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Biol. Chem.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28634232/">The phosphocarrier protein HPr of the bacterial phosphotransferase system globally regulates energy metabolism by directly interacting with multiple enzymes in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28634232/" class="summary-link">
    <div class="article-style">
      <p>The histidine-phosphorylatable phosphocarrier protein (HPr) is an essential component of the sugar-transporting phosphotransferase system (PTS) in many bacteria. Recent interactome findings suggested that HPr interacts with several carbohydrate-metabolizing enzymes, but whether HPr plays a regulatory role was unclear. Here, we provide evidence that HPr interacts with a large number of proteins in Escherichia coli We demonstrate HPr-dependent allosteric regulation of the activities of pyruvate kinase (PykF, but not PykA), phosphofructokinase (PfkB, but not PfkA), glucosamine-6-phosphate deaminase (NagB), and adenylate kinase (Adk). HPr is either phosphorylated on a histidyl residue (HPr-P) or non-phosphorylated (HPr). PykF is activated only by non-phosphorylated HPr, which decreases the PykF Khalf for phosphoenolpyruvate by 10-fold (from 3.5 to 0.36 mm), thus influencing glycolysis. PfkB activation by HPr, but not by HPr-P, resulted from a decrease in the Khalf for fructose-6-P, which likely influences both gluconeogenesis and glycolysis. Moreover, NagB activation by HPr was important for the utilization of amino sugars, and allosteric inhibition of Adk activity by HPr-P, but not by HPr, allows HPr to regulate the cellular energy charge coordinately with glycolysis. These observations suggest that HPr serves as a directly interacting global regulator of carbon and energy metabolism and probably of other physiological processes in enteric bacteria.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28634232/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1074/jbc.M117.795294" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Kate Sokolina</span>, <span >Saranya Kittanakom</span>, <span >Jamie Snider</span>, <span >Max Kotlyar</span>, <span >Pascal Maurice</span>, <span >Jorge Gandía</span>, <span >Abla Benleulmi-Chaachoua</span>, <span >Kenjiro Tadagaki</span>, <span >Atsuro Oishi</span>, <span >Victoria Wong</span>, <span >Ramy H Malty</span>, <span >Viktor Deineko</span>, <span >Hiroyuki Aoki</span>, <span >Shahreen Amin</span>, <span >Zhong Yao</span>, <span >Xavier Morató</span>, <span >David Otasek</span>, <span >Hiroyuki Kobayashi</span>, <span >Javier Menendez</span>, <span >Daniel Auerbach</span>, <span >Stephane Angers</span>, <span >Natasa Pržulj</span>, <span >Michel Bouvier</span>, <span >Mohan Babu</span>, <span >Francisco Ciruela</span>, <span >Ralf Jockers</span>, <span >Igor Jurisica</span>, <span >Igor Stagljar</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Syst. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28298427/">Systematic protein-protein interaction mapping for clinically relevant human GPCRs.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28298427/" class="summary-link">
    <div class="article-style">
      <p>G-protein-coupled receptors (GPCRs) are the largest family of integral membrane receptors with key roles in regulating signaling pathways targeted by therapeutics, but are difficult to study using existing proteomics technologies due to their complex biochemical features. To obtain a global view of GPCR-mediated signaling and to identify novel components of their pathways, we used a modified membrane yeast two-hybrid (MYTH) approach and identified interacting partners for 48 selected full-length human ligand-unoccupied GPCRs in their native membrane environment. The resulting GPCR interactome connects 686 proteins by 987 unique interactions, including 299 membrane proteins involved in a diverse range of cellular functions. To demonstrate the biological relevance of the GPCR interactome, we validated novel interactions of the GPR37, serotonin 5-HT4d, and adenosine ADORA2A receptors. Our data represent the first large-scale interactome mapping for human GPCRs and provide a valuable resource for the analysis of signaling pathways involving this druggable family of integral membrane proteins.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28298427/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.15252/msb.20167430" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Bahram Samanfar</span>, <span >Kristina Shostak</span>, <span >Houman Moteshareie</span>, <span >Maryam Hajikarimlou</span>, <span >Sarah Shaikho</span>, <span >Katayoun Omidi</span>, <span >Mohsen Hooshyar</span>, <span >Daniel Burnside</span>, <span >Imelda Galván Márquez</span>, <span >Tom Kazmirchuk</span>, <span >Thet Naing</span>, <span >Paula Ludovico</span>, <span >Anna York-Lyon</span>, <span >Kama Szereszewski</span>, <span >Cindy Leung</span>, <span >Jennifer Yixin Jin</span>, <span >Rami Megarbane</span>, <span >Myron L Smith</span>, <span >Mohan Babu</span>, <span >Martin Holcik</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PeerJ</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29158977/">The sensitivity of the yeast, Saccharomyces cerevisiae, to acetic acid is influenced by DOM34 and RPL36A.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29158977/" class="summary-link">
    <div class="article-style">
      <p>The presence of acetic acid during industrial alcohol fermentation reduces the yield of fermentation by imposing additional stress on the yeast cells. The biology of cellular responses to stress has been a subject of vigorous investigations. Although much has been learned, details of some of these responses remain poorly understood. Members of heat shock chaperone HSP proteins have been linked to acetic acid and heat shock stress responses in yeast. Both acetic acid and heat shock have been identified to trigger different cellular responses including reduction of global protein synthesis and induction of programmed cell death. Yeast HSC82 and HSP82 code for two important heat shock proteins that together account for 1-2% of total cellular proteins. Both proteins have been linked to responses to acetic acid and heat shock. In contrast to the overall rate of protein synthesis which is reduced, the expression of HSC82 and HSP82 is induced in response to acetic acid stress. In the current study we identified two yeast genes DOM34 and RPL36A that are linked to acetic acid and heat shock sensitivity. We investigated the influence of these genes on the expression of HSP proteins. Our observations suggest that Dom34 and RPL36A influence translation in a CAP-independent manner.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29158977/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.7717/peerj.4037" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Dong Jun</span>, <span >Zoran Minic</span>, <span >Supriya V Bhat</span>, <span >Elizabeth M Vanderlinde</span>, <span >Chris K Yost</span>, <span >Mohan Babu</span>, <span >Tanya E S Dahms</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Front Microbiol</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29354107/">Metabolic Adaptation of a C-Terminal Protease A-Deficient Rhizobium leguminosarum in Response to Loss of Nutrient Transport.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29354107/" class="summary-link">
    <div class="article-style">
      <p>Post-translational modification expands the functionality of the proteome beyond genetic encoding, impacting many cellular processes. Cleavage of the carboxyl terminus is one of the many different ways proteins can be modified for functionality. Gel-electrophoresis and mass spectrometric-based techniques were used to identify proteins impacted by deficiency of a C-terminal protease, CtpA, in Rhizobium leguminosarum bv. viciae 3841. Predicted CtpA substrates from 2D silver stained gels were predominantly outer membrane and transport proteins. Proteins with altered abundance in the wild type and ctpA (RL4692) mutant, separated by 2D difference gel electrophoresis, were selected for analysis by mass spectrometry. Of those identified, 9 were the periplasmic solute-binding components of ABC transporters, 5 were amino acid metabolic enzymes, 2 were proteins involved in sulfur metabolism, and 1 each was related to carbon metabolism, protein folding and signal transduction. Alterations to ABC-binding-cassette transporters, nutrient uptake efficiency and to amino acid metabolism indicated an impact on amino acid metabolism and transport for the ctpA mutant, which was validated by measured amino acid levels.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29354107/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmicb.2017.02617" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Zhong Yao</span>, <span >Katelyn Darowski</span>, <span >Nicole St-Denis</span>, <span >Victoria Wong</span>, <span >Fabian Offensperger</span>, <span >Annabel Villedieu</span>, <span >Shahreen Amin</span>, <span >Ramy Malty</span>, <span >Hiroyuki Aoki</span>, <span >Hongbo Guo</span>, <span >Yang Xu</span>, <span >Caterina Iorio</span>, <span >Max Kotlyar</span>, <span >Andrew Emili</span>, <span >Igor Jurisica</span>, <span >Benjamin G Neel</span>, <span >Mohan Babu</span>, <span >Anne-Claude Gingras</span>, <span >Igor Stagljar</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Cell</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-28065597/">A Global Analysis of the Receptor Tyrosine Kinase-Protein Phosphatase Interactome.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-28065597/" class="summary-link">
    <div class="article-style">
      <p>Receptor tyrosine kinases (RTKs) and protein phosphatases comprise protein families that play crucial roles in cell signaling. We used two protein-protein interaction (PPI) approaches, the membrane yeast two-hybrid (MYTH) and the mammalian membrane two-hybrid (MaMTH), to map the PPIs between human RTKs and phosphatases. The resulting RTK-phosphatase interactome reveals a considerable number of previously unidentified interactions and suggests specific roles for different phosphatase families. Additionally, the differential PPIs of some protein tyrosine phosphatases (PTPs) and their mutants suggest diverse mechanisms of these PTPs in the regulation of RTK signaling. We further found that PTPRH and PTPRB directly dephosphorylate EGFR and repress its downstream signaling. By contrast, PTPRA plays a dual role in EGFR signaling: besides facilitating EGFR dephosphorylation, it enhances downstream ERK signaling by activating SRC. This comprehensive RTK-phosphatase interactome study provides a broad and deep view of RTK signaling.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-28065597/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.molcel.2016.12.004" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Alla Gagarinova</span>, <span >Geordie Stewart</span>, <span >Bahram Samanfar</span>, <span >Sadhna Phanse</span>, <span >Carl A White</span>, <span >Hiroyuki Aoki</span>, <span >Viktor Deineko</span>, <span >Natalia Beloglazova</span>, <span >Alexander F Yakunin</span>, <span >Ashkan Golshani</span>, <span >Eric D Brown</span>, <span >Mohan Babu</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Rep</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-27732863/">Systematic Genetic Screens Reveal the Dynamic Global Functional Organization of the Bacterial Translation Machinery.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-27732863/" class="summary-link">
    <div class="article-style">
      <p>Bacterial protein synthesis is an essential, conserved, and environmentally responsive process. Yet, many of its components and dependencies remain unidentified. To address this gap, we used quantitative synthetic genetic arrays to map functional relationships among &gt;48,000 gene pairs in Escherichia coli under four culture conditions differing in temperature and nutrient availability. The resulting data provide global functional insights into the roles and associations of genes, pathways, and processes important for efficient translation, growth, and environmental adaptation. We predict and independently verify the requirement of unannotated genes for normal translation, including a previously unappreciated role of YhbY in 30S biogenesis. Dynamic changes in the patterns of genetic dependencies across the four growth conditions and data projections onto other species reveal overarching functional and evolutionary pressures impacting the translation system and bacterial fitness, underscoring the utility of systematic screens for investigating protein synthesis, adaptation, and evolution.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-27732863/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.celrep.2016.09.040" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ednalise Santiago</span>, <span >Pearl Akamine</span>, <span >Jamie Snider</span>, <span >Victoria Wong</span>, <span >Matthew Jessulat</span>, <span >Viktor Deineko</span>, <span >Alla Gagarinova</span>, <span >Hiroyuki Aoki</span>, <span >Zoran Minic</span>, <span >Sadhna Phanse</span>, <span >Andrea San Antonio</span>, <span >Luis A Cubano</span>, <span >Brian C Rymond</span>, <span >Mohan Babu</span>, <span >Igor Stagljar</span>, <span >Jose R Rodriguez-Medina</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>G3 (Bethesda)</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26921299/">Novel Interactome of Saccharomyces cerevisiae Myosin Type II Identified by a Modified Integrated Membrane Yeast Two-Hybrid (iMYTH) Screen.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26921299/" class="summary-link">
    <div class="article-style">
      <p>Nonmuscle myosin type II (Myo1p) is required for cytokinesis in the budding yeast Saccharomyces cerevisiae Loss of Myo1p activity has been associated with growth abnormalities and enhanced sensitivity to osmotic stress, making it an appealing antifungal therapeutic target. The Myo1p tail-only domain was previously reported to have functional activity equivalent to the full-length Myo1p whereas the head-only domain did not. Since Myo1p tail-only constructs are biologically active, the tail domain must have additional functions beyond its previously described role in myosin dimerization or trimerization. The identification of new Myo1p-interacting proteins may shed light on the other functions of the Myo1p tail domain. To identify novel Myo1p-interacting proteins, and determine if Myo1p can serve as a scaffold to recruit proteins to the bud neck during cytokinesis, we used the integrated split-ubiquitin membrane yeast two-hybrid (iMYTH) system. Myo1p was iMYTH-tagged at its C-terminus, and screened against both cDNA and genomic prey libraries to identify interacting proteins. Control experiments showed that the Myo1p-bait construct was appropriately expressed, and that the protein colocalized to the yeast bud neck. Thirty novel Myo1p-interacting proteins were identified by iMYTH. Eight proteins were confirmed by coprecipitation (Ape2, Bzz1, Fba1, Pdi1, Rpl5, Tah11, and Trx2) or mass spectrometry (AP-MS) (Abp1). The novel Myo1p-interacting proteins identified come from a range of different processes, including cellular organization and protein synthesis. Actin assembly/disassembly factors such as the SH3 domain protein Bzz1 and the actin-binding protein Abp1 represent likely Myo1p interactions during cytokinesis.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26921299/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1534/g3.115.026609" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mauro Fasano</span>, <span >Tiziana Alberio</span>, <span >Mohan Babu</span>, <span >Emma Lundberg</span>, <span >Andrea Urbani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>EuPA Open Proteom</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-29900096/">Towards a functional definition of the mitochondrial human proteome.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-29900096/" class="summary-link">
    <div class="article-style">
      <p>The mitochondrial human proteome project (mt-HPP) was initiated by the Italian HPP group as a part of both the chromosome-centric initiative (C-HPP) and the ⿿biology and disease driven⿿ initiative (B/D-HPP). In recent years several reports highlighted how mitochondrial biology and disease are regulated by specific interactions with non-mitochondrial proteins. Thus, it is of great relevance to extend our present view of the mitochondrial proteome not only to those proteins that are encoded by or transported to mitochondria, but also to their interactors that take part in mitochondria functionality. Here, we propose a graphical representation of the functional mitochondrial proteome by retrieving mitochondrial proteins from the NeXtProt database and adding to the network their interactors as annotated in the IntAct database. Notably, the network may represent a reference to map all the proteins that are currently being identified in mitochondrial proteomics studies.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-29900096/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.euprot.2016.01.004" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Sadhna Phanse</span>, <span >Cuihong Wan</span>, <span >Blake Borgeson</span>, <span >Fan Tu</span>, <span >Kevin Drew</span>, <span >Greg Clark</span>, <span >Xuejian Xiong</span>, <span >Olga Kagan</span>, <span >Julian Kwan</span>, <span >Alexandr Bezginov</span>, <span >Kyle Chessman</span>, <span >Swati Pal</span>, <span >Graham Cromar</span>, <span >Ophelia Papoulas</span>, <span >Zuyao Ni</span>, <span >Daniel R Boutz</span>, <span >Snejana Stoilova</span>, <span >Pierre C Havugimana</span>, <span >Xinghua Guo</span>, <span >Ramy H Malty</span>, <span >Mihail Sarov</span>, <span >Jack Greenblatt</span>, <span >Mohan Babu</span>, <span >W Brent Derry</span>, <span >Elisabeth R Tillier</span>, <span >John B Wallingford</span>, <span >John Parkinson</span>, <span >Edward M Marcotte</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Data Brief</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26870755/">Proteome-wide dataset supporting the study of ancient metazoan macromolecular complexes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26870755/" class="summary-link">
    <div class="article-style">
      <p>Our analysis examines the conservation of multiprotein complexes among metazoa through use of high resolution biochemical fractionation and precision mass spectrometry applied to soluble cell extracts from 5 representative model organisms Caenorhabditis elegans, Drosophila melanogaster, Mus musculus, Strongylocentrotus purpuratus, and Homo sapiens. The interaction network obtained from the data was validated globally in 4 distant species (Xenopus laevis, Nematostella vectensis, Dictyostelium discoideum, Saccharomyces cerevisiae) and locally by targeted affinity-purification experiments. Here we provide details of our massive set of supporting biochemical fractionation data available via ProteomeXchange (PXD002319-PXD002328), PPIs via BioGRID (185267); and interaction network projections via (<a href="http://metazoa.med.utoronto.ca">http://metazoa.med.utoronto.ca</a>) made fully accessible to allow further exploration. The datasets here are related to the research article on metazoan macromolecular complexes in Nature [1].</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26870755/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.dib.2015.11.062" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ashwani Kumar</span>, <span >Natalia Beloglazova</span>, <span >Cedoljub Bundalovic-Torma</span>, <span >Sadhna Phanse</span>, <span >Viktor Deineko</span>, <span >Alla Gagarinova</span>, <span >Gabriel Musso</span>, <span >James Vlasblom</span>, <span >Sofia Lemak</span>, <span >Mohsen Hooshyar</span>, <span >Zoran Minic</span>, <span >Omar Wagih</span>, <span >Roberto Mosca</span>, <span >Patrick Aloy</span>, <span >Ashkan Golshani</span>, <span >John Parkinson</span>, <span >Andrew Emili</span>, <span >Alexander F Yakunin</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Rep</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26774489/">Conditional Epistatic Interaction Maps Reveal Global Functional Rewiring of Genome Integrity Pathways in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26774489/" class="summary-link">
    <div class="article-style">
      <p>As antibiotic resistance is increasingly becoming a public health concern, an improved understanding of the bacterial DNA damage response (DDR), which is commonly targeted by antibiotics, could be of tremendous therapeutic value. Although the genetic components of the bacterial DDR have been studied extensively in isolation, how the underlying biological pathways interact functionally remains unclear. Here, we address this by performing systematic, unbiased, quantitative synthetic genetic interaction (GI) screens and uncover widespread changes in the GI network of the entire genomic integrity apparatus of Escherichia coli under standard and DNA-damaging growth conditions. The GI patterns of untreated cultures implicated two previously uncharacterized proteins (YhbQ and YqgF) as nucleases, whereas reorganization of the GI network after DNA damage revealed DDR roles for both annotated and uncharacterized genes. Analyses of pan-bacterial conservation patterns suggest that DDR mechanisms and functional relationships are near universal, highlighting a modular and highly adaptive genomic stress response.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26774489/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.celrep.2015.12.060" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Jingjing Li</span>, <span >Zhihai Ma</span>, <span >Minyi Shi</span>, <span >Ramy H Malty</span>, <span >Hiroyuki Aoki</span>, <span >Zoran Minic</span>, <span >Sadhna Phanse</span>, <span >Ke Jin</span>, <span >Dennis P Wall</span>, <span >Zhaolei Zhang</span>, <span >Alexander E Urban</span>, <span >Joachim Hallmayer</span>, <span >Mohan Babu</span>, <span >Michael Snyder</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell Syst</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26949739/">Identification of Human Neuronal Protein Complexes Reveals Biochemical Activities and Convergent Mechanisms of Action in Autism Spectrum Disorders.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26949739/" class="summary-link">
    <div class="article-style">
      <p>The prevalence of autism spectrum disorders (ASDs) is rapidly growing, yet its molecular basis is poorly understood. We used a systems approach in which ASD candidate genes were mapped onto the ubiquitous human protein complexes and the resulting complexes were characterized. The studies revealed the role of histone deacetylases (HDAC1/2) in regulating the expression of ASD orthologs in the embryonic mouse brain. Proteome-wide screens for the co-complexed subunits with HDAC1 and six other key ASD proteins in neuronal cells revealed a protein interaction network, which displayed preferential expression in fetal brain development, exhibited increased deleterious mutations in ASD cases, and were strongly regulated by FMRP and MECP2 causal for Fragile X and Rett syndromes, respectively. Overall, our study reveals molecular components in ASD, suggests a shared mechanism between the syndromic and idiopathic forms of ASDs, and provides a systems framework for analyzing complex human diseases.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26949739/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.cels.2015.11.002" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Cuihong Wan</span>, <span >Blake Borgeson</span>, <span >Sadhna Phanse</span>, <span >Fan Tu</span>, <span >Kevin Drew</span>, <span >Greg Clark</span>, <span >Xuejian Xiong</span>, <span >Olga Kagan</span>, <span >Julian Kwan</span>, <span >Alexandr Bezginov</span>, <span >Kyle Chessman</span>, <span >Swati Pal</span>, <span >Graham Cromar</span>, <span >Ophelia Papoulas</span>, <span >Zuyao Ni</span>, <span >Daniel R Boutz</span>, <span >Snejana Stoilova</span>, <span >Pierre C Havugimana</span>, <span >Xinghua Guo</span>, <span >Ramy H Malty</span>, <span >Mihail Sarov</span>, <span >Jack Greenblatt</span>, <span >Mohan Babu</span>, <span >W Brent Derry</span>, <span >Elisabeth R Tillier</span>, <span >John B Wallingford</span>, <span >John Parkinson</span>, <span >Edward M Marcotte</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/BabuLab/publication/pmid-26344197/">
      <img src="/BabuLab/publication/pmid-26344197/featured_hu276eacabe4ef1efd12758e1ca6caf910_74939_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="Panorama of ancient metazoan macromolecular complexes.">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26344197/">Panorama of ancient metazoan macromolecular complexes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26344197/" class="summary-link">
    <div class="article-style">
      <p>Macromolecular complexes are essential to conserved biological processes, but their prevalence across animals is unclear. By combining extensive biochemical fractionation with quantitative mass spectrometry, here we directly examined the composition of soluble multiprotein complexes among diverse metazoan models. Using an integrative approach, we generated a draft conservation map consisting of more than one million putative high-confidence co-complex interactions for species with fully sequenced genomes that encompasses functional modules present broadly across all extant animals. Clustering reveals a spectrum of conservation, ranging from ancient eukaryotic assemblies that have probably served cellular housekeeping roles for at least one billion years, ancestral complexes that have accrued contemporary components, and rarer metazoan innovations linked to multicellularity. We validated these projections by independent co-fractionation experiments in evolutionarily distant species, affinity purification and functional analyses. The comprehensiveness, centrality and modularity of these reconstructed interactomes reflect their fundamental mechanistic importance and adaptive value to animal cell systems.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26344197/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nature14877" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Matthew Jessulat</span>, <span >Ramy H Malty</span>, <span >Diem-Hang Nguyen-Tran</span>, <span >Viktor Deineko</span>, <span >Hiroyuki Aoki</span>, <span >James Vlasblom</span>, <span >Katayoun Omidi</span>, <span >Ke Jin</span>, <span >Zoran Minic</span>, <span >Mohsen Hooshyar</span>, <span >Daniel Burnside</span>, <span >Bahram Samanfar</span>, <span >Sadhna Phanse</span>, <span >Tanya Freywald</span>, <span >Bhanu Prasad</span>, <span >Zhaolei Zhang</span>, <span >Franco Vizeacoumar</span>, <span >Nevan J Krogan</span>, <span >Andrew Freywald</span>, <span >Ashkan Golshani</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Cell. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25963654/">Spindle Checkpoint Factors Bub1 and Bub2 Promote DNA Double-Strand Break Repair by Nonhomologous End Joining.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25963654/" class="summary-link">
    <div class="article-style">
      <p>The nonhomologous end-joining (NHEJ) pathway is essential for the preservation of genome integrity, as it efficiently repairs DNA double-strand breaks (DSBs). Previous biochemical and genetic investigations have indicated that, despite the importance of this pathway, the entire complement of genes regulating NHEJ remains unknown. To address this, we employed a plasmid-based NHEJ DNA repair screen in budding yeast (Saccharomyces cerevisiae) using 369 putative nonessential DNA repair-related components as queries. Among the newly identified genes associated with NHEJ deficiency upon disruption are two spindle assembly checkpoint kinases, Bub1 and Bub2. Both observation of resulting phenotypes and chromatin immunoprecipitation demonstrated that Bub1 and -2, either alone or in combination with cell cycle regulators, are recruited near the DSB, where phosphorylated Rad53 or H2A accumulates. Large-scale proteomic analysis of Bub kinases phosphorylated in response to DNA damage identified previously unknown kinase substrates on Tel1 S/T-Q sites. Moreover, Bub1 NHEJ function appears to be conserved in mammalian cells. 53BP1, which influences DSB repair by NHEJ, colocalizes with human BUB1 and is recruited to the break sites. Thus, while Bub is not a core component of NHEJ machinery, our data support its dual role in mitotic exit and promotion of NHEJ repair in yeast and mammals.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25963654/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/MCB.00007-15" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ekaterina Kuznetsova</span>, <span >Boguslaw Nocek</span>, <span >Greg Brown</span>, <span >Kira S Makarova</span>, <span >Robert Flick</span>, <span >Yuri I Wolf</span>, <span >Anna Khusnutdinova</span>, <span >Elena Evdokimova</span>, <span >Ke Jin</span>, <span >Kemin Tan</span>, <span >Andrew D Hanson</span>, <span >Ghulam Hasnain</span>, <span >Rémi Zallot</span>, <span >Valérie de Crécy-Lagard</span>, <span >Mohan Babu</span>, <span >Alexei Savchenko</span>, <span >Andrzej Joachimiak</span>, <span >Aled M Edwards</span>, <span >Eugene V Koonin</span>, <span >Alexander F Yakunin</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Biol. Chem.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26071590/">Functional Diversity of Haloacid Dehalogenase Superfamily Phosphatases from Saccharomyces cerevisiae: BIOCHEMICAL, STRUCTURAL, AND EVOLUTIONARY INSIGHTS.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26071590/" class="summary-link">
    <div class="article-style">
      <p>The haloacid dehalogenase (HAD)-like enzymes comprise a large superfamily of phosphohydrolases present in all organisms. The Saccharomyces cerevisiae genome encodes at least 19 soluble HADs, including 10 uncharacterized proteins. Here, we biochemically characterized 13 yeast phosphatases from the HAD superfamily, which includes both specific and promiscuous enzymes active against various phosphorylated metabolites and peptides with several HADs implicated in detoxification of phosphorylated compounds and pseudouridine. The crystal structures of four yeast HADs provided insight into their active sites, whereas the structure of the YKR070W dimer in complex with substrate revealed a composite substrate-binding site. Although the S. cerevisiae and Escherichia coli HADs share low sequence similarities, the comparison of their substrate profiles revealed seven phosphatases with common preferred substrates. The cluster of secondary substrates supporting significant activity of both S. cerevisiae and E. coli HADs includes 28 common metabolites that appear to represent the pool of potential activities for the evolution of novel HAD phosphatases. Evolution of novel substrate specificities of HAD phosphatases shows no strict correlation with sequence divergence. Thus, evolution of the HAD superfamily combines the conservation of the overall substrate pool and the substrate profiles of some enzymes with remarkable biochemical and structural flexibility of other superfamily members.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26071590/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1074/jbc.M115.657916" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mandy Hiu Yi Lam</span>, <span >Jamie Snider</span>, <span >Monique Rehal</span>, <span >Victoria Wong</span>, <span >Farzaneh Aboualizadeh</span>, <span >Luka Drecun</span>, <span >Olivia Wong</span>, <span >Bellal Jubran</span>, <span >Meirui Li</span>, <span >Mehrab Ali</span>, <span >Matthew Jessulat</span>, <span >Viktor Deineko</span>, <span >Rachel Miller</span>, <span >Mid eum Lee</span>, <span >Hay-Oak Park</span>, <span >Alan Davidson</span>, <span >Mohan Babu</span>, <span >Igor Stagljar</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Mol. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25644660/">A Comprehensive Membrane Interactome Mapping of Sho1p Reveals Fps1p as a Novel Key Player in the Regulation of the HOG Pathway in S. cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25644660/" class="summary-link">
    <div class="article-style">
      <p>Sho1p, an integral membrane protein, plays a vital role in the high-osmolarity glycerol (HOG) mitogen-activated protein kinase pathway in the yeast Saccharomyces cerevisiae. Activated under conditions of high osmotic stress, it interacts with other HOG pathway proteins to mediate cell signaling events, ensuring that yeast cells can adapt and remain viable. In an attempt to further understand how the function of Sho1p is regulated through its protein-protein interactions (PPIs), we identified 49 unique Sho1p PPIs through the use of membrane yeast two-hybrid (MYTH), an assay specifically suited to identify PPIs of full-length integral membrane proteins in their native membrane environment. Secondary validation by literature search, or two complementary PPI assays, confirmed 80% of these interactions, resulting in a high-quality Sho1p interactome. This set of putative PPIs included both previously characterized interactors, along with a large subset of interactors that have not been previously identified as binding to Sho1p. The SH3 domain of Sho1p was found to be important for binding to many of these interactors. One particular novel interactor of interest is the glycerol transporter Fps1p, which was shown to require the SH3 domain of Sho1p for binding via its N-terminal soluble regulatory domain. Furthermore, we found that Fps1p is involved in the positive regulation of Sho1p function and plays a role in the phosphorylation of the downstream kinase Hog1p. This study represents the largest membrane interactome analysis of Sho1p to date and complements past studies on the HOG pathway by increasing our understanding of Sho1p regulation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25644660/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.jmb.2015.01.016" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Bjorn D M Bean</span>, <span >Michael Davey</span>, <span >Jamie Snider</span>, <span >Matthew Jessulat</span>, <span >Viktor Deineko</span>, <span >Matthew Tinney</span>, <span >Igor Stagljar</span>, <span >Mohan Babu</span>, <span >Elizabeth Conibear</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Biol. Cell</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25609093/">Rab5-family guanine nucleotide exchange factors bind retromer and promote its recruitment to endosomes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25609093/" class="summary-link">
    <div class="article-style">
      <p>The retromer complex facilitates the sorting of integral membrane proteins from the endosome to the late Golgi. In mammalian cells, the efficient recruitment of retromer to endosomes requires the lipid phosphatidylinositol 3-phosphate (PI3P) as well as Rab5 and Rab7 GTPases. However, in yeast, the role of Rabs in recruiting retromer to endosomes is less clear. We identified novel physical interactions between retromer and the Saccharomyces cerevisiae VPS9-domain Rab5-family guanine nucleotide exchange factors (GEFs) Muk1 and Vps9. Furthermore, we identified a new yeast VPS9 domain-containing protein, VARP-like 1 (Vrl1), which is related to the human VARP protein. All three VPS9 domain-containing proteins show localization to endosomes, and the presence of any one of them is necessary for the endosomal recruitment of retromer. We find that expression of an active VPS9-domain protein is required for correct localization of the phosphatidylinositol 3-kinase Vps34 and the production of endosomal PI3P. These results suggest that VPS9 GEFs promote retromer recruitment by establishing PI3P-enriched domains at the endosomal membrane. The interaction of retromer with distinct VPS9 GEFs could thus link GEF-dependent regulatory inputs to the temporal or spatial coordination of retromer assembly or function.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25609093/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1091/mbc.E14-08-1281" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ke Jin</span>, <span >Gabriel Musso</span>, <span >James Vlasblom</span>, <span >Matthew Jessulat</span>, <span >Viktor Deineko</span>, <span >Jacopo Negroni</span>, <span >Roberto Mosca</span>, <span >Ramy Malty</span>, <span >Diem-Hang Nguyen-Tran</span>, <span >Hiroyuki Aoki</span>, <span >Zoran Minic</span>, <span >Tanya Freywald</span>, <span >Sadhna Phanse</span>, <span >Qian Xiang</span>, <span >Andrew Freywald</span>, <span >Patrick Aloy</span>, <span >Zhaolei Zhang</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Proteome Res.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25546499/">Yeast mitochondrial protein-protein interactions reveal diverse complexes and disease-relevant functional relationships.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25546499/" class="summary-link">
    <div class="article-style">
      <p>Although detailed, focused, and mechanistic analyses of associations among mitochondrial proteins (MPs) have identified their importance in varied biological processes, a systematic understanding of how MPs function in concert both with one another and with extra-mitochondrial proteins remains incomplete. Consequently, many questions regarding the role of mitochondrial dysfunction in the development of human disease remain unanswered. To address this, we compiled all existing mitochondrial physical interaction data for over 1200 experimentally defined yeast MPs and, through bioinformatic analysis, identified hundreds of heteromeric MP complexes having extensive associations both within and outside the mitochondria. We provide support for these complexes through structure prediction analysis, morphological comparisons of deletion strains, and protein co-immunoprecipitation. The integration of these MP complexes with reported genetic interaction data reveals substantial crosstalk between MPs and non-MPs and identifies novel factors in endoplasmic reticulum-mitochondrial organization, membrane structure, and mitochondrial lipid homeostasis. More than one-third of these MP complexes are conserved in humans, with many containing members linked to clinical pathologies, enabling us to identify genes with putative disease function through guilt-by-association. Although still remaining incomplete, existing mitochondrial interaction data suggests that the relevant molecular machinery is modular, yet highly integrated with non-mitochondrial processes.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25546499/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1021/pr501148q" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >James Vlasblom</span>, <span >Khalid Zuberi</span>, <span >Harold Rodriguez</span>, <span >Roland Arnold</span>, <span >Alla Gagarinova</span>, <span >Viktor Deineko</span>, <span >Ashwani Kumar</span>, <span >Elisa Leung</span>, <span >Kamran Rizzolo</span>, <span >Bahram Samanfar</span>, <span >Luke Chang</span>, <span >Sadhna Phanse</span>, <span >Ashkan Golshani</span>, <span >Jack F Greenblatt</span>, <span >Walid A Houry</span>, <span >Andrew Emili</span>, <span >Quaid Morris</span>, <span >Gary Bader</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25316676/">Novel function discovery with GeneMANIA: a new integrated resource for gene function prediction in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25316676/" class="summary-link">
    <div class="article-style">
      <p>The model bacterium Escherichia coli is among the best studied prokaryotes, yet nearly half of its proteins are still of unknown biological function. This is despite a wealth of available large-scale physical and genetic interaction data. To address this, we extended the GeneMANIA function prediction web application developed for model eukaryotes to support E.coli.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25316676/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btu671" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Viktor Deineko</span>, <span >Ashwani Kumar</span>, <span >James Vlasblom</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Adv. Exp. Med. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-26621466/">Quantitative and Systems-Based Approaches for Deciphering Bacterial Membrane Interactome and Gene Function.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-26621466/" class="summary-link">
    <div class="article-style">
      <p>High-throughput genomic and proteomic methods provide a concise description of the molecular constituents of a cell, whereas systems biology strives to understand the way these components function as a whole. Recent developments, such as genome editing technologies and protein epitope-tagging coupled with high-sensitivity mass-spectrometry, allow systemic studies to be performed at an unprecedented scale. Available methods can be successfully applied to various goals, both expanding fundamental knowledge and solving applied problems. In this review, we discuss the present state and future of bacterial cell envelope interactomics, with a specific focus on host-pathogen interactions and drug target discovery. Both experimental and computational methods will be outlined together with examples of their practical implementation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-26621466/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-3-319-23603-2_8" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ramy H Malty</span>, <span >Matthew Jessulat</span>, <span >Ke Jin</span>, <span >Gabriel Musso</span>, <span >James Vlasblom</span>, <span >Sadhna Phanse</span>, <span >Zhaolei Zhang</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Proteome Res.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25367773/">Mitochondrial targets for pharmacological intervention in human disease.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25367773/" class="summary-link">
    <div class="article-style">
      <p>Over the past several years, mitochondrial dysfunction has been linked to an increasing number of human illnesses, making mitochondrial proteins (MPs) an ever more appealing target for therapeutic intervention. With 20% of the mitochondrial proteome (312 of an estimated 1500 MPs) having known interactions with small molecules, MPs appear to be highly targetable. Yet, despite these targeted proteins functioning in a range of biological processes (including induction of apoptosis, calcium homeostasis, and metabolism), very few of the compounds targeting MPs find clinical use. Recent work has greatly expanded the number of proteins known to localize to the mitochondria and has generated a considerable increase in MP 3D structures available in public databases, allowing experimental screening and in silico prediction of mitochondrial drug targets on an unprecedented scale. Here, we summarize the current literature on clinically active drugs that target MPs, with a focus on how existing drug targets are distributed across biochemical pathways and organelle substructures. Also, we examine current strategies for mitochondrial drug discovery, focusing on genetic, proteomic, and chemogenomic assays, and relevant model systems. As cell models and screening techniques improve, MPs appear poised to emerge as relevant targets for a wide range of complex human diseases, an eventuality that can be expedited through systematic analysis of MP function.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25367773/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1021/pr500813f" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Odette Allonby</span>, <span >Amr M El Zawily</span>, <span >Tanya Freywald</span>, <span >Darrell D Mousseau</span>, <span >Jennifer Chlan</span>, <span >Deborah Anderson</span>, <span >Alexandre Benmerah</span>, <span >Vishaldeep Sidhu</span>, <span >Mohan Babu</span>, <span >John DeCoteau</span>, <span >Andrew Freywald</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell. Signal.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25152371/">Ligand stimulation induces clathrin- and Rab5-dependent downregulation of the kinase-dead EphB6 receptor preceded by the disruption of EphB6-Hsp90 interaction.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25152371/" class="summary-link">
    <div class="article-style">
      <p>Ligand-induced internalisation and subsequent downregulation of receptor tyrosine kinases (RTKs) serve to determine biological outputs of their signalling. Intrinsically kinase-deficient RTKs control a variety of biological responses, however, the mechanism of their downregulation is not well understood and its analysis is focused exclusively on the ErbB3 receptor. The Eph group of RTKs is represented by the EphA and EphB subclasses. Each bears one kinase-inactive member, EphA10 and EphB6, respectively, suggesting an important role for these molecules in the Eph signalling network. While EphB6 effects on cell behaviour have been assessed, the mechanism of its downregulation remains elusive. Our work reveals that EphB6 and its kinase-active relative, and signalling partner, EphB4, are downregulated in a similar manner in response to their common ligand, ephrin-B2. Following stimulation, both receptors are internalised through clathrin-coated pits and are degraded in lysosomes. Their targeting for lysosomal degradation relies on the activity of an early endosome regulator, the Rab5 GTPase, as this process is inhibited in the presence of a Rab5 dominant-negative mutant. EphB6 also interacts with the Hsp90 chaperone and EphB6 downregulation is preceded by their rapid dissociation. Moreover, the inhibition of Hsp90 results in EphB6 degradation, mimicking its ligand-induced downregulation. These processes appear to rely on overlapping mechanisms, since Hsp90 inhibition does not significantly enhance ligand-induced EphB6 elimination. Taken together, our observations define a novel mechanism for intrinsically kinase-deficient RTK downregulation and support an intriguing model, where Hsp90 dissociation acts as a trigger for ligand-induced receptor removal.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25152371/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.cellsig.2014.08.007" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Andrew Schoenrock</span>, <span >Bahram Samanfar</span>, <span >Sylvain Pitre</span>, <span >Mohsen Hooshyar</span>, <span >Ke Jin</span>, <span >Charles A Phillips</span>, <span >Hui Wang</span>, <span >Sadhna Phanse</span>, <span >Katayoun Omidi</span>, <span >Yuan Gui</span>, <span >Md Alamgir</span>, <span >Alex Wong</span>, <span >Fredrik Barrenäs</span>, <span >Mohan Babu</span>, <span >Mikael Benson</span>, <span >Michael A Langston</span>, <span >James R Green</span>, <span >Frank Dehne</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-25492630/">Efficient prediction of human protein-protein interactions at a global scale.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-25492630/" class="summary-link">
    <div class="article-style">
      <p>Our knowledge of global protein-protein interaction (PPI) networks in complex organisms such as humans is hindered by technical limitations of current methods.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-25492630/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s12859-014-0383-1" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >James Vlasblom</span>, <span >Ke Jin</span>, <span >Sandy Kassir</span>, <span >Mohan Babu</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J Proteomics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24262152/">Exploring mitochondrial system properties of neurodegenerative diseases through interactome mapping.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24262152/" class="summary-link">
    <div class="article-style">
      <p>Mitochondria are double membraned, dynamic organelles that are required for a large number of cellular processes, and defects in their function have emerged as causative factors for a growing number of human disorders and are highly associated with cancer, metabolic, and neurodegenerative (ND) diseases. Biochemical and genetic investigations have uncovered small numbers of candidate mitochondrial proteins (MPs) involved in ND disease, but given the diversity of processes affected by MP function and the difficulty of detecting interactions involving these proteins, many more likely remain unknown. However, high-throughput proteomic and genomic approaches developed in genetically tractable model prokaryotes and lower eukaryotes have proven to be effective tools for querying the physical (protein-protein) and functional (gene-gene) relationships between diverse types of proteins, including cytosolic and membrane proteins. In this review, we highlight how experimental and computational approaches developed recently by our group and others can be effectively used towards elucidating the mitochondrial interactome in an unbiased and systematic manner to uncover network-based connections. We discuss how the knowledge from the resulting interaction networks can effectively contribute towards the identification of new mitochondrial disease gene candidates, and thus further clarify the role of mitochondrial biology and the complex etiologies of ND disease.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24262152/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.jprot.2013.11.008" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Bahram Samanfar</span>, <span >Le Hoa Tan</span>, <span >Kristina Shostak</span>, <span >Firoozeh Chalabian</span>, <span >Zongbin Wu</span>, <span >Md Alamgir</span>, <span >Noor Sunba</span>, <span >Daniel Burnside</span>, <span >Katayoun Omidi</span>, <span >Mohsen Hooshyar</span>, <span >Imelda Galván Márquez</span>, <span >Matthew Jessulat</span>, <span >Myron L Smith</span>, <span >Mohan Babu</span>, <span >Ali Azizi</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol Biosyst</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24535059/">A global investigation of gene deletion strains that affect premature stop codon bypass in yeast, Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24535059/" class="summary-link">
    <div class="article-style">
      <p>Protein biosynthesis is an orderly process that requires a balance between rate and accuracy. To produce a functional product, the fidelity of this process has to be maintained from start to finish. In order to systematically identify genes that affect stop codon bypass, three expression plasmids, pUKC817, pUKC818 and pUKC819, were integrated into the yeast non-essential loss-of-function gene array (5000 strains). These plasmids contain three different premature stop codons (UAA, UGA and UAG, respectively) within the LacZ expression cassette. A fourth plasmid, pUKC815 that carries the native LacZ gene was used as a control. Transformed strains were subjected to large-scale β-galactosidase lift assay analysis to evaluate production of β-galactosidase for each gene deletion strain. In this way 84 potential candidate genes that affect stop codon bypass were identified. Three candidate genes, OLA1, BSC2, and YNL040W, were further investigated, and were found to be important for cytoplasmic protein biosynthesis.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24535059/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1039/c3mb70501c" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Seesandra V Rajagopala</span>, <span >Patricia Sikorski</span>, <span >Ashwani Kumar</span>, <span >Roberto Mosca</span>, <span >James Vlasblom</span>, <span >Roland Arnold</span>, <span >Jonathan Franca-Koh</span>, <span >Suman B Pakala</span>, <span >Sadhna Phanse</span>, <span >Arnaud Ceol</span>, <span >Roman Häuser</span>, <span >Gabriella Siszler</span>, <span >Stefan Wuchty</span>, <span >Andrew Emili</span>, <span >Mohan Babu</span>, <span >Patrick Aloy</span>, <span >Rembert Pieper</span>, <span >Peter Uetz</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat. Biotechnol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24561554/">The binary protein-protein interaction landscape of Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24561554/" class="summary-link">
    <div class="article-style">
      <p>Efforts to map the Escherichia coli interactome have identified several hundred macromolecular complexes, but direct binary protein-protein interactions (PPIs) have not been surveyed on a large scale. Here we performed yeast two-hybrid screens of 3,305 baits against 3,606 preys (∼70% of the E. coli proteome) in duplicate to generate a map of 2,234 interactions, which approximately doubles the number of known binary PPIs in E. coli. Integration of binary PPI and genetic-interaction data revealed functional dependencies among components involved in cellular processes, including envelope integrity, flagellum assembly and protein quality control. Many of the binary interactions that we could map in multiprotein complexes were informative regarding internal topology of complexes and indicated that interactions in complexes are substantially more conserved than those interactions connecting different complexes. This resource will be useful for inferring bacterial gene function and provides a draft reference of the basic physical wiring network of this evolutionarily important model microbe.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24561554/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nbt.2831" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Roland Arnold</span>, <span >Cedoljub Bundalovic-Torma</span>, <span >Alla Gagarinova</span>, <span >Keith S Wong</span>, <span >Ashwani Kumar</span>, <span >Geordie Stewart</span>, <span >Bahram Samanfar</span>, <span >Hiroyuki Aoki</span>, <span >Omar Wagih</span>, <span >James Vlasblom</span>, <span >Sadhna Phanse</span>, <span >Krunal Lad</span>, <span >Angela Yeou Hsiung Yu</span>, <span >Christopher Graham</span>, <span >Ke Jin</span>, <span >Eric Brown</span>, <span >Ashkan Golshani</span>, <span >Philip Kim</span>, <span >Gabriel Moreno-Hagelsieb</span>, <span >Jack Greenblatt</span>, <span >Walid A Houry</span>, <span >John Parkinson</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS Genet.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24586182/">Quantitative genome-wide genetic interaction screens reveal global epistatic relationships of protein complexes in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24586182/" class="summary-link">
    <div class="article-style">
      <p>Large-scale proteomic analyses in Escherichia coli have documented the composition and physical relationships of multiprotein complexes, but not their functional organization into biological pathways and processes. Conversely, genetic interaction (GI) screens can provide insights into the biological role(s) of individual gene and higher order associations. Combining the information from both approaches should elucidate how complexes and pathways intersect functionally at a systems level. However, such integrative analysis has been hindered due to the lack of relevant GI data. Here we present a systematic, unbiased, and quantitative synthetic genetic array screen in E. coli describing the genetic dependencies and functional cross-talk among over 600,000 digenic mutant combinations. Combining this epistasis information with putative functional modules derived from previous proteomic data and genomic context-based methods revealed unexpected associations, including new components required for the biogenesis of iron-sulphur and ribosome integrity, and the interplay between molecular chaperones and proteases. We find that functionally-linked genes co-conserved among γ-proteobacteria are far more likely to have correlated GI profiles than genes with divergent patterns of evolution. Overall, examining bacterial GIs in the context of protein complexes provides avenues for a deeper mechanistic understanding of core microbial systems.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24586182/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pgen.1004120" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Andrey M Grishin</span>, <span >Maia Cherney</span>, <span >Deborah H Anderson</span>, <span >Sadhna Phanse</span>, <span >Mohan Babu</span>, <span >Miroslaw Cygler</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Structure</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24373767/">NleH defines a new family of bacterial effector kinases.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24373767/" class="summary-link">
    <div class="article-style">
      <p>Upon host cell infection, pathogenic Escherichia coli hijacks host cellular processes with the help of 20-60 secreted effector proteins that subvert cellular processes to create an environment conducive to bacterial survival. The NleH effector kinases manipulate the NF-κB pathway and prevent apoptosis. They show low sequence similarity to human regulatory kinases and contain two domains, the N-terminal, likely intrinsically unfolded, and a C-terminal kinase-like domain. We show that these effectors autophosphorylate on sites located predominantly in the N-terminal segment. The kinase domain displays a minimal kinase fold, but lacks an activation loop and the GHI subdomain. Nevertheless, all catalytically important residues are conserved. ATP binding proceeds with minimal structural rearrangements. The NleH structure is the first for the bacterial effector kinases family. NleHs and their homologous effector kinases form a new kinase family within the cluster of eukaryotic-like kinases that includes also Rio, Bud32, and KdoK families.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24373767/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.str.2013.11.006" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Keith S Wong</span>, <span >Jamie D Snider</span>, <span >Chris Graham</span>, <span >Jack F Greenblatt</span>, <span >Andrew Emili</span>, <span >Mohan Babu</span>, <span >Walid A Houry</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS ONE</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24454883/">The MoxR ATPase RavA and its cofactor ViaA interact with the NADH:ubiquinone oxidoreductase I in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24454883/" class="summary-link">
    <div class="article-style">
      <p>MoxR ATPases are widespread throughout bacteria and archaea. The experimental evidence to date suggests that these proteins have chaperone-like roles in facilitating the maturation of dedicated protein complexes that are functionally diverse. In Escherichia coli, the MoxR ATPase RavA and its putative cofactor ViaA are found to exist in early stationary-phase cells at 37 °C at low levels of about 350 and 90 molecules per cell, respectively. Both proteins are predominantly localized to the cytoplasm, but ViaA was also unexpectedly found to localize to the cell membrane. Whole genome microarrays and synthetic lethality studies both indicated that RavA-ViaA are genetically linked to Fe-S cluster assembly and specific respiratory pathways. Systematic analysis of mutant strains of ravA and viaA indicated that RavA-ViaA sensitizes cells to sublethal concentrations of aminoglycosides. Furthermore, this effect was dependent on RavA&rsquo;s ATPase activity, and on the presence of specific subunits of NADH:ubiquinone oxidoreductase I (Nuo Complex, or Complex I). Importantly, both RavA and ViaA were found to physically interact with specific Nuo subunits. We propose that RavA-ViaA facilitate the maturation of the Nuo complex.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24454883/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0085529" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Katayoun Omidi</span>, <span >Mohsen Hooshyar</span>, <span >Matthew Jessulat</span>, <span >Bahram Samanfar</span>, <span >Megan Sanders</span>, <span >Daniel Burnside</span>, <span >Sylvain Pitre</span>, <span >Andrew Schoenrock</span>, <span >Jianhua Xu</span>, <span >Mohan Babu</span>, <span >Ashkan Golshani</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS ONE</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24498054/">Phosphatase complex Pph3/Psy2 is involved in regulation of efficient non-homologous end-joining pathway in the yeast Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24498054/" class="summary-link">
    <div class="article-style">
      <p>One of the main mechanisms for double stranded DNA break (DSB) repair is through the non-homologous end-joining (NHEJ) pathway. Using plasmid and chromosomal repair assays, we showed that deletion mutant strains for interacting proteins Pph3p and Psy2p had reduced efficiencies in NHEJ. We further observed that this activity of Pph3p and Psy2p appeared linked to cell cycle Rad53p and Chk1p checkpoint proteins. Pph3/Psy2 is a phosphatase complex, which regulates recovery from the Rad53p DNA damage checkpoint. Overexpression of Chk1p checkpoint protein in a parallel pathway to Rad53p compensated for the deletion of PPH3 or PSY2 in a chromosomal repair assay. Double mutant strains Δpph3/Δchk1 and Δpsy2/Δchk1 showed additional reductions in the efficiency of plasmid repair, compared to both single deletions which is in agreement with the activity of Pph3p and Psy2p in a parallel pathway to Chk1p. Genetic interaction analyses also supported a role for Pph3p and Psy2p in DNA damage repair, the NHEJ pathway, as well as cell cycle progression. Collectively, we report that the activity of Pph3p and Psy2p further connects NHEJ repair to cell cycle progression.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24498054/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0087248" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Franco J Vizeacoumar</span>, <span >Roland Arnold</span>, <span >Frederick S Vizeacoumar</span>, <span >Megha Chandrashekhar</span>, <span >Alla Buzina</span>, <span >Jordan T F Young</span>, <span >Julian H M Kwan</span>, <span >Azin Sayad</span>, <span >Patricia Mero</span>, <span >Steffen Lawo</span>, <span >Hiromasa Tanaka</span>, <span >Kevin R Brown</span>, <span >Anastasia Baryshnikova</span>, <span >Anthony B Mak</span>, <span >Yaroslav Fedyshyn</span>, <span >Yadong Wang</span>, <span >Glauber C Brito</span>, <span >Dahlia Kasimer</span>, <span >Taras Makhnevych</span>, <span >Troy Ketela</span>, <span >Alessandro Datti</span>, <span >Mohan Babu</span>, <span >Andrew Emili</span>, <span >Laurence Pelletier</span>, <span >Jeff Wrana</span>, <span >Zev Wainberg</span>, <span >Philip M Kim</span>, <span >Robert Rottapel</span>, <span >Catherine A O&#39;Brien</span>, <span >Brenda Andrews</span>, <span >Charles Boone</span>, <span >Jason Moffat</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Syst. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-24104479/">A negative genetic interaction map in isogenic cancer cell lines reveals cancer cell vulnerabilities.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-24104479/" class="summary-link">
    <div class="article-style">
      <p>Improved efforts are necessary to define the functional product of cancer mutations currently being revealed through large-scale sequencing efforts. Using genome-scale pooled shRNA screening technology, we mapped negative genetic interactions across a set of isogenic cancer cell lines and confirmed hundreds of these interactions in orthogonal co-culture competition assays to generate a high-confidence genetic interaction network of differentially essential or differential essentiality (DiE) genes. The network uncovered examples of conserved genetic interactions, densely connected functional modules derived from comparative genomics with model systems data, functions for uncharacterized genes in the human genome and targetable vulnerabilities. Finally, we demonstrate a general applicability of DiE gene signatures in determining genetic dependencies of other non-isogenic cancer cell lines. For example, the PTEN(-/-) DiE genes reveal a signature that can preferentially classify PTEN-dependent genotypes across a series of non-isogenic cell lines derived from the breast, pancreas and ovarian cancers. Our reference network suggests that many cancer vulnerabilities remain to be discovered through systematic derivation of a network of differentially essential genes in an isogenic cancer cell model.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-24104479/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/msb.2013.54" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Jamie Snider</span>, <span >Asad Hanif</span>, <span >Mid Eum Lee</span>, <span >Ke Jin</span>, <span >Analyn R Yu</span>, <span >Chris Graham</span>, <span >Matthew Chuk</span>, <span >Dunja Damjanovic</span>, <span >Marta Wierzbicka</span>, <span >Priscilla Tang</span>, <span >Dina Balderes</span>, <span >Victoria Wong</span>, <span >Matthew Jessulat</span>, <span >Katelyn D Darowski</span>, <span >Bryan-Joseph San Luis</span>, <span >Igor Shevelev</span>, <span >Stephen L Sturley</span>, <span >Charles Boone</span>, <span >Jack F Greenblatt</span>, <span >Zhaolei Zhang</span>, <span >Christian M Paumi</span>, <span >Mohan Babu</span>, <span >Hay-Oak Park</span>, <span >Susan Michaelis</span>, <span >Igor Stagljar</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat. Chem. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-23831759/">Mapping the functional yeast ABC transporter interactome.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-23831759/" class="summary-link">
    <div class="article-style">
      <p>ATP-binding cassette (ABC) transporters are a ubiquitous class of integral membrane proteins of immense clinical interest because of their strong association with human disease and pharmacology. To improve our understanding of these proteins, we used membrane yeast two-hybrid technology to map the protein interactome of all of the nonmitochondrial ABC transporters in the model organism Saccharomyces cerevisiae and combined this data with previously reported yeast ABC transporter interactions in the BioGRID database to generate a comprehensive, integrated &lsquo;interactome&rsquo;. We show that ABC transporters physically associate with proteins involved in an unexpectedly diverse range of functions. We specifically examine the importance of the physical interactions of ABC transporters in both the regulation of one another and in the modulation of proteins involved in zinc homeostasis. The interaction network presented here will be a powerful resource for increasing our fundamental understanding of the cellular role and regulation of ABC transporters.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-23831759/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nchembio.1293" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Martin Graef</span>, <span >Jonathan R Friedman</span>, <span >Christopher Graham</span>, <span >Mohan Babu</span>, <span >Jodi Nunnari</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Biol. Cell</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-23904270/">ER exit sites are physical and functional core autophagosome biogenesis components.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-23904270/" class="summary-link">
    <div class="article-style">
      <p>Autophagy is a central homeostasis and stress response pathway conserved in all eukaryotes. One hallmark of autophagy is the de novo formation of autophagosomes. These double-membrane vesicular structures form around and deliver cargo for degradation by the vacuole/lysosome. Where and how autophagosomes form are outstanding questions. Here we show, using proteomic, cytological, and functional analyses, that autophagosomes are spatially, physically, and functionally linked to endoplasmic reticulum exit sites (ERES), which are specialized regions of the endoplasmic reticulum where COPII transport vesicles are generated. Our data demonstrate that ERES are core autophagosomal biogenesis components whose function is required for the hierarchical assembly of the autophagy machinery immediately downstream of the Atg1 kinase complex at phagophore assembly sites.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-23904270/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1091/mbc.E13-07-0381" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Michael E Donaldson</span>, <span >Shaowu Meng</span>, <span >Alla Gagarinova</span>, <span >Mohan Babu</span>, <span >Scott C Lambie</span>, <span >Alexander A Swiadek</span>, <span >Barry J Saville</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Fungal Genet. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-23973481/">Investigating the Ustilago maydis/Zea mays pathosystem: transcriptional responses and novel functional aspects of a fungal calcineurin regulatory B subunit.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-23973481/" class="summary-link">
    <div class="article-style">
      <p>The sustainable control of basidiomycete biotrophic plant pathogenesis requires an understanding of host responses to infection, as well as the identification and functional analysis of fungal genes involved in disease development. The creation and analysis of a suppressive subtractive hybridization (SSH) cDNA library from Ustilago maydis-infected Zea mays seedlings enabled the identification of fungal and plant genes expressed during disease development, and uncovered new insights into the interactions of this model system. Candidate U. maydis pathogenesis genes were identified by using the current SSH cDNA library analysis, and by knowledge generated from previous cDNA microarray and comparative genomic analyses. These identifications were supported by the independent determination of transcript level changes in different cell-types and during pathogenic development. The basidiomycete specific um01632, the highly in planta expressed um03046 (zig1), and the calcineurin regulatory B subunit (um10226, cnb1), were chosen for deletion experiments. um01632 and zig1 mutants showed no difference in morphology and did not have a statistically significant impact on pathogenesis. cnb1 mutants had a distinct cell division phenotype and reduced virulence in seedling assays. Infections with reciprocal wild-type×Δcnb1 haploid strain crosses revealed that the wild-type allele was unable to fully compensate for the lack of a second cnb1 allele. This haploinsufficiency was undetected in other fungal cnb1 mutational analyses. The reported data improves U. maydis genome annotation and expands on the current understanding of pathogenesis genes in this model basidiomycete.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-23973481/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.fgb.2013.08.006" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Alla Gagarinova</span>, <span >Mohan Babu</span>, <span >Jack Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November 2012
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J Vis Exp</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-23168417/">Mapping bacterial functional networks and pathways in Escherichia Coli using synthetic genetic arrays.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-23168417/" class="summary-link">
    <div class="article-style">
      <p>Phenotypes are determined by a complex series of physical (e.g. protein-protein) and functional (e.g. gene-gene or genetic) interactions (GI)(1). While physical interactions can indicate which bacterial proteins are associated as complexes, they do not necessarily reveal pathway-level functional relationships1. GI screens, in which the growth of double mutants bearing two deleted or inactivated genes is measured and compared to the corresponding single mutants, can illuminate epistatic dependencies between loci and hence provide a means to query and discover novel functional relationships(2). Large-scale GI maps have been reported for eukaryotic organisms like yeast(3-7), but GI information remains sparse for prokaryotes(8), which hinders the functional annotation of bacterial genomes. To this end, we and others have developed high-throughput quantitative bacterial GI screening methods(9, 10). Here, we present the key steps required to perform quantitative E. coli Synthetic Genetic Array (eSGA) screening procedure on a genome-scale(9), using natural bacterial conjugation and homologous recombination to systemically generate and measure the fitness of large numbers of double mutants in a colony array format. Briefly, a robot is used to transfer, through conjugation, chloramphenicol (Cm) - marked mutant alleles from engineered Hfr (High frequency of recombination) &lsquo;donor strains&rsquo; into an ordered array of kanamycin (Kan) - marked F- recipient strains. Typically, we use loss-of-function single mutants bearing non-essential gene deletions (e.g. the &lsquo;Keio&rsquo; collection(11)) and essential gene hypomorphic mutations (i.e. alleles conferring reduced protein expression, stability, or activity(9, 12, 13)) to query the functional associations of non-essential and essential genes, respectively. After conjugation and ensuing genetic exchange mediated by homologous recombination, the resulting double mutants are selected on solid medium containing both antibiotics. After outgrowth, the plates are digitally imaged and colony sizes are quantitatively scored using an in-house automated image processing system(14). GIs are revealed when the growth rate of a double mutant is either significantly better or worse than expected(9). Aggravating (or negative) GIs often result between loss-of-function mutations in pairs of genes from compensatory pathways that impinge on the same essential process(2). Here, the loss of a single gene is buffered, such that either single mutant is viable. However, the loss of both pathways is deleterious and results in synthetic lethality or sickness (i.e. slow growth). Conversely, alleviating (or positive) interactions can occur between genes in the same pathway or protein complex(2) as the deletion of either gene alone is often sufficient to perturb the normal function of the pathway or complex such that additional perturbations do not reduce activity, and hence growth, further. Overall, systematically identifying and analyzing GI networks can provide unbiased, global maps of the functional relationships between large numbers of genes, from which pathway-level information missed by other approaches can be inferred(9).</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-23168417/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3791/4056" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Olga Kagan</span>, <span >Hongbo Guo</span>, <span >Jack Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November 2012
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J Vis Exp</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-23168686/">Identification of protein complexes in Escherichia coli using sequential peptide affinity purification in combination with tandem mass spectrometry.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-23168686/" class="summary-link">
    <div class="article-style">
      <p>Since most cellular processes are mediated by macromolecular assemblies, the systematic identification of protein-protein interactions (PPI) and the identification of the subunit composition of multi-protein complexes can provide insight into gene function and enhance understanding of biological systems(1, 2). Physical interactions can be mapped with high confidence vialarge-scale isolation and characterization of endogenous protein complexes under near-physiological conditions based on affinity purification of chromosomally-tagged proteins in combination with mass spectrometry (APMS). This approach has been successfully applied in evolutionarily diverse organisms, including yeast, flies, worms, mammalian cells, and bacteria(1-6). In particular, we have generated a carboxy-terminal Sequential Peptide Affinity (SPA) dual tagging system for affinity-purifying native protein complexes from cultured gram-negative Escherichia coli, using genetically-tractable host laboratory strains that are well-suited for genome-wide investigations of the fundamental biology and conserved processes of prokaryotes(1, 2, 7). Our SPA-tagging system is analogous to the tandem affinity purification method developed originally for yeast(8, 9), and consists of a calmodulin binding peptide (CBP) followed by the cleavage site for the highly specific tobacco etch virus (TEV) protease and three copies of the FLAG epitope (3X FLAG), allowing for two consecutive rounds of affinity enrichment. After cassette amplification, sequence-specific linear PCR products encoding the SPA-tag and a selectable marker are integrated and expressed in frame as carboxy-terminal fusions in a DY330 background that is induced to transiently express a highly efficient heterologous bacteriophage lambda recombination system(10). Subsequent dual-step purification using calmodulin and anti-FLAG affinity beads enables the highly selective and efficient recovery of even low abundance protein complexes from large-scale cultures. Tandem mass spectrometry is then used to identify the stably co-purifying proteins with high sensitivity (low nanogram detection limits). Here, we describe detailed step-by-step procedures we commonly use for systematic protein tagging, purification and mass spectrometry-based analysis of soluble protein complexes from E. coli, which can be scaled up and potentially tailored to other bacterial species, including certain opportunistic pathogens that are amenable to recombineering. The resulting physical interactions can often reveal interesting unexpected components and connections suggesting novel mechanistic links. Integration of the PPI data with alternate molecular association data such as genetic (gene-gene) interactions and genomic-context (GC) predictions can facilitate elucidation of the global molecular organization of multi-protein complexes within biological pathways. The networks generated for E. coli can be used to gain insight into the functional architecture of orthologous gene products in other microbes for which functional annotations are currently lacking.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-23168686/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3791/4057" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >James Vlasblom</span>, <span >Shuye Pu</span>, <span >Xinghua Guo</span>, <span >Chris Graham</span>, <span >Björn D M Bean</span>, <span >Helen E Burston</span>, <span >Franco J Vizeacoumar</span>, <span >Jamie Snider</span>, <span >Sadhna Phanse</span>, <span >Vincent Fong</span>, <span >Yuen Yi C Tam</span>, <span >Michael Davey</span>, <span >Olha Hnatshak</span>, <span >Navgeet Bajaj</span>, <span >Shamanta Chandran</span>, <span >Thanuja Punna</span>, <span >Constantine Christopolous</span>, <span >Victoria Wong</span>, <span >Analyn Yu</span>, <span >Gouqing Zhong</span>, <span >Joyce Li</span>, <span >Igor Stagljar</span>, <span >Elizabeth Conibear</span>, <span >Shoshana J Wodak</span>, <span >Andrew Emili</span>, <span >Jack F Greenblatt</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2012
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-22940862/">Interaction landscape of membrane-protein complexes in Saccharomyces cerevisiae.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-22940862/" class="summary-link">
    <div class="article-style">
      <p>Macromolecular assemblies involving membrane proteins (MPs) serve vital biological roles and are prime drug targets in a variety of diseases. Large-scale affinity purification studies of soluble-protein complexes have been accomplished for diverse model organisms, but no global characterization of MP-complex membership has been described so far. Here we report a complete survey of 1,590 putative integral, peripheral and lipid-anchored MPs from Saccharomyces cerevisiae, which were affinity purified in the presence of non-denaturing detergents. The identities of the co-purifying proteins were determined by tandem mass spectrometry and subsequently used to derive a high-confidence physical interaction map encompassing 1,726 membrane protein-protein interactions and 501 putative heteromeric complexes associated with the various cellular membrane systems. Our analysis reveals unexpected physical associations underlying the membrane biology of eukaryotes and delineates the global topological landscape of the membrane interactome.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-22940862/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nature11354" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Pierre C Havugimana</span>, <span >G Traver Hart</span>, <span >Tamás Nepusz</span>, <span >Haixuan Yang</span>, <span >Andrei L Turinsky</span>, <span >Zhihua Li</span>, <span >Peggy I Wang</span>, <span >Daniel R Boutz</span>, <span >Vincent Fong</span>, <span >Sadhna Phanse</span>, <span >Mohan Babu</span>, <span >Stephanie A Craig</span>, <span >Pingzhao Hu</span>, <span >Cuihong Wan</span>, <span >James Vlasblom</span>, <span >Vaqaar-un-Nisa Dar</span>, <span >Alexandr Bezginov</span>, <span >Gregory W Clark</span>, <span >Gabriel C Wu</span>, <span >Shoshana J Wodak</span>, <span >Elisabeth R M Tillier</span>, <span >Alberto Paccanaro</span>, <span >Edward M Marcotte</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2012
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Cell</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-22939629/">A census of human soluble protein complexes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-22939629/" class="summary-link">
    <div class="article-style">
      <p>Cellular processes often depend on stable physical associations between proteins. Despite recent progress, knowledge of the composition of human protein complexes remains limited. To close this gap, we applied an integrative global proteomic profiling approach, based on chromatographic separation of cultured human cell extracts into more than one thousand biochemical fractions that were subsequently analyzed by quantitative tandem mass spectrometry, to systematically identify a network of 13,993 high-confidence physical interactions among 3,006 stably associated soluble human proteins. Most of the 622 putative protein complexes we report are linked to core biological processes and encompass both candidate disease genes and unannotated proteins to inform on mechanism. Strikingly, whereas larger multiprotein assemblies tend to be more extensively annotated and evolutionarily conserved, human protein complexes with five or fewer subunits are far more likely to be functionally unannotated or restricted to vertebrates, suggesting more recent functional innovations.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-22939629/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.cell.2012.08.011" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >J Javier Díaz-Mejía</span>, <span >James Vlasblom</span>, <span >Alla Gagarinova</span>, <span >Sadhna Phanse</span>, <span >Chris Graham</span>, <span >Fouad Yousif</span>, <span >Huiming Ding</span>, <span >Xuejian Xiong</span>, <span >Anaies Nazarians-Armavil</span>, <span >Md Alamgir</span>, <span >Mehrab Ali</span>, <span >Oxana Pogoutse</span>, <span >Asaf Pe&#39;er</span>, <span >Roland Arnold</span>, <span >Magali Michaut</span>, <span >John Parkinson</span>, <span >Ashkan Golshani</span>, <span >Chris Whitfield</span>, <span >Shoshana J Wodak</span>, <span >Gabriel Moreno-Hagelsieb</span>, <span >Jack F Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS Genet.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-22125496/">Genetic interaction maps in Escherichia coli reveal functional crosstalk among cell envelope biogenesis pathways.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-22125496/" class="summary-link">
    <div class="article-style">
      <p>As the interface between a microbe and its environment, the bacterial cell envelope has broad biological and clinical significance. While numerous biosynthesis genes and pathways have been identified and studied in isolation, how these intersect functionally to ensure envelope integrity during adaptive responses to environmental challenge remains unclear. To this end, we performed high-density synthetic genetic screens to generate quantitative functional association maps encompassing virtually the entire cell envelope biosynthetic machinery of Escherichia coli under both auxotrophic (rich medium) and prototrophic (minimal medium) culture conditions. The differential patterns of genetic interactions detected among &gt; 235,000 digenic mutant combinations tested reveal unexpected condition-specific functional crosstalk and genetic backup mechanisms that ensure stress-resistant envelope assembly and maintenance. These networks also provide insights into the global systems connectivity and dynamic functional reorganization of a universal bacterial structure that is both broadly conserved among eubacteria (including pathogens) and an important target.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-22125496/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pgen.1002377" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Hiroyuki Aoki</span>, <span >Wasimul Q Chowdhury</span>, <span >Alla Gagarinova</span>, <span >Chris Graham</span>, <span >Sadhna Phanse</span>, <span >Ben Laliberte</span>, <span >Noor Sunba</span>, <span >Matthew Jessulat</span>, <span >Ashkan Golshani</span>, <span >Andrew Emili</span>, <span >Jack F Greenblatt</span>, <span >M Clelia Ganoza</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS ONE</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-21556145/">Ribosome-dependent ATPase interacts with conserved membrane protein in Escherichia coli to modulate protein synthesis and oxidative phosphorylation.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-21556145/" class="summary-link">
    <div class="article-style">
      <p>Elongation factor RbbA is required for ATP-dependent deacyl-tRNA release presumably after each peptide bond formation; however, there is no information about the cellular role. Proteomic analysis in Escherichia coli revealed that RbbA reciprocally co-purified with a conserved inner membrane protein of unknown function, YhjD. Both proteins are also physically associated with the 30S ribosome and with members of the lipopolysaccharide transport machinery. Genome-wide genetic screens of rbbA and yhjD deletion mutants revealed aggravating genetic interactions with mutants deficient in the electron transport chain. Cells lacking both rbbA and yhjD exhibited reduced cell division, respiration and global protein synthesis as well as increased sensitivity to antibiotics targeting the ETC and the accuracy of protein synthesis. Our results suggest that RbbA appears to function together with YhjD as part of a regulatory network that impacts bacterial oxidative phosphorylation and translation efficiency.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-21556145/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0018510" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Alla Gagarinova</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Methods Mol. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-21877280/">Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-21877280/" class="summary-link">
    <div class="article-style">
      <p>Cellular processes are carried out through a series of molecular interactions. Various experimental approaches can be used to investigate these functional relationships on a large-scale. Recently, the power of investigating biological systems from the perspective of genetic (gene-gene, or epistatic) interactions has been evidenced by the ability to elucidate novel functional relationships. Examples of functionally related genes include genes that buffer each other&rsquo;s function or impinge on the same biological process. Genetic interactions have traditionally been investigated in bacteria by combining pairs of mutations (for example, gene deletions) and assessing deviation of the phenotype of each double mutant from an expected neutral (or no interaction) phenotype. Fitness is a particularly convenient phenotype to measure: when the double mutant grows faster or slower than expected, the two mutated genes are said to show alleviating or aggravating interactions, respectively. The most commonly used neutral model assumes that the fitness of the double mutant is equal to the product of individual single mutant fitness. A striking genetic interaction is exemplified by the loss of two nonessential genes that buffer each other in performing an essential biological function: deleting only one of these genes produces no detectable fitness defect; however, loss of both genes simultaneously results in systems failure, leading to synthetic sickness or lethality. Systematic large-scale genetic interaction screens have been used to generate functional maps for model eukaryotic organisms, such as yeast, to describe the functional organization of gene products into pathways and protein complexes within a cell. They also reveal the modular arrangement and cross-talk of pathways and complexes within broader functional neighborhoods (Dixon et al. Annu Rev Genet 43:601-625, 2009). Here, we present a high-throughput quantitative Escherichia coli synthetic genetic array (eSGA) screening procedure, which we developed to systematically infer genetic interactions by scoring growth defects among large numbers of double mutants in a classic gram-negative bacterium. The eSGA method exploits the rapid colony growth, ease of genetic manipulation, and natural efficient genetic exchange via conjugation of laboratory E. coli strains. Replica pinning is used to grow and mate arrayed sets of single-gene mutant strains as well as to select double mutants en mass. Strain fitness, which is used as the eSGA readout, is quantified by the digital imaging of the plates and subsequent measuring and comparing single and double mutant colony sizes. While eSGA can be used to screen select mutants to probe the functions of individual genes; using eSGA more broadly to collect genetic interaction data for many combinations of genes can help reconstruct a functional interaction network to reveal novel links and components of biological pathways as well as unexpected connections between pathways. A variety of bacterial systems can be investigated, wherein the genes impinge on a essential biological process (e.g., cell wall assembly, ribosome biogenesis, chromosome replication) that are of interest from the perspective of drug development (Babu et al. Mol Biosyst 12:1439-1455, 2009). We also show how genetic interactions generated by high-throughput eSGA screens can be validated by manual small-scale genetic crosses and by genetic complementation and gene rescue experiments.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-21877280/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-1-61779-276-2_7" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Alla Gagarinova</span>, <span >Jack Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Methods Mol. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-21815091/">Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-21815091/" class="summary-link">
    <div class="article-style">
      <p>Cellular processes are carried out through a series of molecular interactions. Various experimental approaches can be used to investigate these functional relationships on a large-scale. Recently, the power of investigating biological systems from the perspective of genetic (gene-gene or epistatic) interactions has been evidenced by the ability to elucidate novel functional relationships. Examples of functionally related genes include genes that buffer each other&rsquo;s function or impinge on the same biological process. Genetic interactions have traditionally been investigated in bacteria by combining pairs of mutations (e.g., gene deletions) and assessing deviation of the phenotype of each double mutant from an expected neutral (or no interaction) phenotype. Fitness is a particularly convenient phenotype to measure: when the double mutant grows faster or slower than expected, the two mutated genes are said to show alleviating or aggravating interactions, respectively. The most commonly used neutral model assumes that the fitness of the double mutant is equal to the product of individual single mutant fitness. A striking genetic interaction is exemplified by the loss of two nonessential genes that buffer each other in performing an essential biological function: deleting only one of these genes produces no detectable fitness defect; however, loss of both genes simultaneously results in systems failure, leading to synthetic sickness or lethality. Systematic large-scale genetic interaction screens have been used to generate functional maps for model eukaryotic organisms, such as yeast, to describe the functional organization of gene products into pathways and protein complexes within a cell. They also reveal the modular arrangement and cross talk of pathways and complexes within broader functional neighborhoods (Dixon et al., Annu Rev Genet 43:601-625, 2009). Here, we present a high-throughput quantitative Escherichia coli Synthetic Genetic Array (eSGA) screening procedure, which we developed to systematically infer genetic interactions by scoring growth defects among large numbers of double mutants in a classic Gram-negative bacterium. The eSGA method exploits the rapid colony growth, ease of genetic manipulation, and natural efficient genetic exchange via conjugation of laboratory E. coli strains. Replica pinning is used to grow and mate arrayed sets of single gene mutant strains and to select double mutants en masse. Strain fitness, which is used as the eSGA readout, is quantified by the digital imaging of the plates and subsequent measuring and comparing single and double mutant colony sizes. While eSGA can be used to screen select mutants to probe the functions of individual genes, using eSGA more broadly to collect genetic interaction data for many combinations of genes can help reconstruct a functional interaction network to reveal novel links and components of biological pathways as well as unexpected connections between pathways. A variety of bacterial systems can be investigated, wherein the genes impinge on a essential biological process (e.g., cell wall assembly, ribosome biogenesis, chromosome replication) that are of interest from the perspective of drug development (Babu et al., Mol Biosyst 12:1439-1455, 2009). We also show how genetic interactions generated by high-throughput eSGA screens can be validated by manual small-scale genetic crosses and by genetic complementation and gene rescue experiments.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-21815091/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-1-61779-197-0_9" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Natalia Beloglazova</span>, <span >Robert Flick</span>, <span >Chris Graham</span>, <span >Tatiana Skarina</span>, <span >Boguslaw Nocek</span>, <span >Alla Gagarinova</span>, <span >Oxana Pogoutse</span>, <span >Greg Brown</span>, <span >Andrew Binkowski</span>, <span >Sadhna Phanse</span>, <span >Andrzej Joachimiak</span>, <span >Eugene V Koonin</span>, <span >Alexei Savchenko</span>, <span >Andrew Emili</span>, <span >Jack Greenblatt</span>, <span >Aled M Edwards</span>, <span >Alexander F Yakunin</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Microbiol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-21219465/">A dual function of the CRISPR-Cas system in bacterial antivirus immunity and DNA repair.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-21219465/" class="summary-link">
    <div class="article-style">
      <p>Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) and the associated proteins (Cas) comprise a system of adaptive immunity against viruses and plasmids in prokaryotes. Cas1 is a CRISPR-associated protein that is common to all CRISPR-containing prokaryotes but its function remains obscure. Here we show that the purified Cas1 protein of Escherichia coli (YgbT) exhibits nuclease activity against single-stranded and branched DNAs including Holliday junctions, replication forks and 5&rsquo;-flaps. The crystal structure of YgbT and site-directed mutagenesis have revealed the potential active site. Genome-wide screens show that YgbT physically and genetically interacts with key components of DNA repair systems, including recB, recC and ruvB. Consistent with these findings, the ygbT deletion strain showed increased sensitivity to DNA damage and impaired chromosomal segregation. Similar phenotypes were observed in strains with deletion of CRISPR clusters, suggesting that the function of YgbT in repair involves interaction with the CRISPRs. These results show that YgbT belongs to a novel, structurally distinct family of nucleases acting on branched DNAs and suggest that, in addition to antiviral immunity, at least some components of the CRISPR-Cas system have a function in DNA repair.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-21219465/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1111/j.1365-2958.2010.07465.x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2010">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Jack F Greenblatt</span>, <span >Andrew Emili</span>, <span >Natalie C J Strynadka</span>, <span >Reinhart A F Reithmeier</span>, <span >Trevor F Moraes</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November 2010
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Structure</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-21070944/">Structure of a SLC26 anion transporter STAS domain in complex with acyl carrier protein: implications for E. coli YchM in fatty acid metabolism.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-21070944/" class="summary-link">
    <div class="article-style">
      <p>Escherichia coli YchM is a member of the SLC26 (SulP) family of anion transporters with an N-terminal membrane domain and a C-terminal cytoplasmic STAS domain. Mutations in human members of the SLC26 family, including their STAS domain, are linked to a number of inherited diseases. Herein, we describe the high-resolution crystal structure of the STAS domain from E. coli YchM isolated in complex with acyl-carrier protein (ACP), an essential component of the fatty acid biosynthesis (FAB) pathway. A genome-wide genetic interaction screen showed that a ychM null mutation is synthetically lethal with mutant alleles of genes (fabBDHGAI) involved in FAB. Endogenous YchM also copurified with proteins involved in fatty acid metabolism. Furthermore, a deletion strain lacking ychM showed altered cellular bicarbonate incorporation in the presence of NaCl and impaired growth at alkaline pH. Thus, identification of the STAS-ACP complex suggests that YchM sequesters ACP to the bacterial membrane linking bicarbonate transport with fatty acid metabolism.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-21070944/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.str.2010.08.015" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2010">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Yuichi Taniguchi</span>, <span >Paul J Choi</span>, <span >Gene-Wei Li</span>, <span >Huiyi Chen</span>, <span >Mohan Babu</span>, <span >Jeremy Hearn</span>, <span >Andrew Emili</span>, <span >X Sunney Xie</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2010
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Science</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-20671182/">Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-20671182/" class="summary-link">
    <div class="article-style">
      <p>Protein and messenger RNA (mRNA) copy numbers vary from cell to cell in isogenic bacterial populations. However, these molecules often exist in low copy numbers and are difficult to detect in single cells. We carried out quantitative system-wide analyses of protein and mRNA expression in individual cells with single-molecule sensitivity using a newly constructed yellow fluorescent protein fusion library for Escherichia coli. We found that almost all protein number distributions can be described by the gamma distribution with two fitting parameters which, at low expression levels, have clear physical interpretations as the transcription rate and protein burst size. At high expression levels, the distributions are dominated by extrinsic noise. We found that a single cell&rsquo;s protein and mRNA copy numbers for any given gene are uncorrelated.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-20671182/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1126/science.1188308" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Gabriel Musso</span>, <span >J Javier Díaz-Mejía</span>, <span >Gareth Butland</span>, <span >Jack F Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2009
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol Biosyst</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-19763343/">Systems-level approaches for identifying and analyzing genetic interaction networks in Escherichia coli and extensions to other prokaryotes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-19763343/" class="summary-link">
    <div class="article-style">
      <p>Molecular interactions define the functional organization of the cell. Epistatic (genetic, or gene-gene) interactions, one of the most informative and commonly encountered forms of functional relationships, are increasingly being used to map process architecture in model eukaryotic organisms. In particular, &lsquo;systems-level&rsquo; screens in yeast and worm aimed at elucidating genetic interaction networks have led to the generation of models describing the global modular organization of gene products and protein complexes within a cell. However, comparable data for prokaryotic organisms have not been available. Given its ease of growth and genetic manipulation, the Gram-negative bacterium Escherichia coli appears to be an ideal model system for performing comprehensive genome-scale examinations of genetic redundancy in bacteria. In this review, we highlight emerging experimental and computational techniques that have been developed recently to examine functional relationships and redundancy in E. coli at a systems-level, and their potential application to prokaryotes in general. Additionally, we have scanned PubMed abstracts and full-text published articles to manually curate a list of approximately 200 previously reported synthetic sick or lethal genetic interactions in E. coli derived from small-scale experimental studies.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-19763343/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1039/B907407d" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Pingzhao Hu</span>, <span >Sarath Chandra Janga</span>, <span >Mohan Babu</span>, <span >J Javier Díaz-Mejía</span>, <span >Gareth Butland</span>, <span >Wenhong Yang</span>, <span >Oxana Pogoutse</span>, <span >Xinghua Guo</span>, <span >Sadhna Phanse</span>, <span >Peter Wong</span>, <span >Shamanta Chandran</span>, <span >Constantine Christopoulos</span>, <span >Anaies Nazarians-Armavil</span>, <span >Negin Karimi Nasseri</span>, <span >Gabriel Musso</span>, <span >Mehrab Ali</span>, <span >Nazila Nazemof</span>, <span >Veronika Eroukova</span>, <span >Ashkan Golshani</span>, <span >Alberto Paccanaro</span>, <span >Jack F Greenblatt</span>, <span >Gabriel Moreno-Hagelsieb</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2009
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-19402753/">Global functional atlas of Escherichia coli encompassing previously uncharacterized proteins.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-19402753/" class="summary-link">
    <div class="article-style">
      <p>One-third of the 4,225 protein-coding genes of Escherichia coli K-12 remain functionally unannotated (orphans). Many map to distant clades such as Archaea, suggesting involvement in basic prokaryotic traits, whereas others appear restricted to E. coli, including pathogenic strains. To elucidate the orphans&rsquo; biological roles, we performed an extensive proteomic survey using affinity-tagged E. coli strains and generated comprehensive genomic context inferences to derive a high-confidence compendium for virtually the entire proteome consisting of 5,993 putative physical interactions and 74,776 putative functional associations, most of which are novel. Clustering of the respective probabilistic networks revealed putative orphan membership in discrete multiprotein complexes and functional modules together with annotated gene products, whereas a machine-learning strategy based on network integration implicated the orphans in specific biological processes. We provide additional experimental evidence supporting orphan participation in protein synthesis, amino acid metabolism, biofilm formation, motility, and assembly of the bacterial cell envelope. This resource provides a &ldquo;systems-wide&rdquo; functional blueprint of a model microbe, with insights into the biological and evolutionary significance of previously uncharacterized proteins.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-19402753/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pbio.1000096" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Nevan J Krogan</span>, <span >Donald E Awrey</span>, <span >Andrew Emili</span>, <span >Jack F Greenblatt</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2009
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Methods Mol. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-19521826/">Systematic characterization of the protein interaction network and protein complexes in Saccharomyces cerevisiae using tandem affinity purification and mass spectrometry.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-19521826/" class="summary-link">
    <div class="article-style">
      <p>Defining protein complexes is a vital aspect of cell biology because cellular processes are often carried out by stable protein complexes and their characterization often provides insights into their function. Accurate identification of the interacting proteins in macromolecular complexes is easiest after purification to near homogeneity. To this end, the tandem affinity purification (TAP) system with subsequent protein identification by high-throughput mass spectrometry was developed (1, 2) to systematically characterize native protein complexes and transient protein interactions under near-physiological conditions. The TAP tag containing two adjacent affinity purification tags (calmodulin-binding peptide and Staphylococcus aureus protein A) separated by a tobacco etch virus (TEV) protease cleavage site is fused with the open reading frame of interest. Using homologous recombination, a fusion library was constructed for the yeast Saccharomyces cerevisiae (3) in which the carboxy-terminal end of each predicted open reading frame is individually tagged in the chromosome so that the resulting fusion proteins are expressed under the control of their natural promoters (3). In this chapter, an optimized protocol for systematic protein purification and subsequent mass spectrometry-based protein identification is described in detail for the protein complexes of S. cerevisiae (4-6).</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-19521826/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-1-59745-540-4_11" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Gareth Butland</span>, <span >Oxana Pogoutse</span>, <span >Joyce Li</span>, <span >Jack F Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2009
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Methods Mol. Biol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-19544035/">Sequential peptide affinity purification system for the systematic isolation and identification of protein complexes from Escherichia coli.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-19544035/" class="summary-link">
    <div class="article-style">
      <p>Biochemical purification of affinity-tagged proteins in combination with mass spectrometry methods is increasingly seen as a cornerstone of systems biology, as it allows for the systematic genome-scale characterization of macromolecular protein complexes, representing demarcated sets of stably interacting protein partners. Accurate and sensitive identification of both the specific and shared polypeptide components of distinct complexes requires purification to near homogeneity. To this end, a sequential peptide affinity (SPA) purification system was developed to enable the rapid and efficient isolation of native Escherichia coli protein complexes (J Proteome Res 3:463-468, 2004). SPA purification makes use of a dual-affinity tag, consisting of three modified FLAG sequences (3X FLAG) and a calmodulin binding peptide (CBP), spaced by a cleavage site for tobacco etch virus (TEV) protease (J Proteome Res 3:463-468, 2004). Using the lambda-phage Red homologous recombination system (PNAS 97:5978-5983, 2000), a DNA cassette, encoding the SPA-tag and a selectable marker flanked by gene-specific targeting sequences, is introduced into a selected locus in the E. coli chromosome so as to create a C-terminal fusion with the protein of interest. This procedure aims for near-endogenous levels of tagged protein production in the recombinant bacteria to avoid spurious, non-specific protein associations (J Proteome Res 3:463-468, 2004). In this chapter, we describe a detailed, optimized protocol for the tagging, purification, and subsequent mass spectrometry-based identification of the subunits of even low-abundance bacterial protein complexes isolated as part of an ongoing large-scale proteomic study in E. coli (Nature 433:531-537, 2005).</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-19544035/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-1-60761-157-8_22" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2009">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Juan Javier Díaz-Mejía</span>, <span >Mohan Babu</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2009
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>FEMS Microbiol. Rev.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-19054114/">Computational and experimental approaches to chart the Escherichia coli cell-envelope-associated proteome and interactome.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-19054114/" class="summary-link">
    <div class="article-style">
      <p>The bacterial cell-envelope consists of a complex arrangement of lipids, proteins and carbohydrates that serves as the interface between a microorganism and its environment or, with pathogens, a human host. Escherichia coli has long been investigated as a leading model system to elucidate the fundamental mechanisms underlying microbial cell-envelope biology. This includes extensive descriptions of the molecular identities, biochemical activities and evolutionary trajectories of integral transmembrane proteins, many of which play critical roles in infectious disease and antibiotic resistance. Strikingly, however, only half of the c. 1200 putative cell-envelope-related proteins of E. coli currently have experimentally attributed functions, indicating an opportunity for discovery. In this review, we summarize the state of the art of computational and proteomic approaches for determining the components of the E. coli cell-envelope proteome, as well as exploring the physical and functional interactions that underlie its biogenesis and functionality. We also provide a comprehensive comparative benchmarking analysis on the performance of different bioinformatic and proteomic methods commonly used to determine the subcellular localization of bacterial proteins.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-19054114/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1111/j.1574-6976.2008.00141.x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2008">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Alla G Gagarinova</span>, <span >Mohan Babu</span>, <span >Vaino Poysa</span>, <span >John H Hill</span>, <span >Aiming Wang</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December 2008
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Virus Res.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-18793685/">Identification and molecular characterization of two naturally occurring Soybean mosaic virus isolates that are closely related but differ in their ability to overcome Rsv4 resistance.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-18793685/" class="summary-link">
    <div class="article-style">
      <p>A naturally occurring Rsv4 resistance-breaking isolate (L-RB) and a closely related non-resistance-breaking isolate (L) of Soybean mosaic virus (SMV) were identified in soybean fields in London, Ontario, Canada. The viral genomes of L and L-RB were completely sequenced. Each isolate has a 9585-nucleotide genome with a single open reading frame encoding a polyprotein of approximately 350 kDa. L-RB and L have a very high sequence similarity (99.6%) at both the nucleotide and amino acid levels. Phylogenetic analysis showed that the two isolates belong to the G2 pathotype. Pathogenicity predictions of all virus/soybean combinations, based on the phylogenetic profile, were confirmed by pathogenicity tests using L and L-RB isolates and soybeans carrying different resistance genes, with an exception that L-RB infected a soybean cultivar carrying Rsv4 resistance. The temporal and spatial proximity of L and L-RB and their high sequence similarity suggest L-RB was likely derived from the SMV-L quasispecies. Recombination analysis did not reveal the evidence of genetic recombination for the emergence of L-RB. Mutations introduced by virus-encoded RNA-dependent RNA polymerase during viral genome replication and selection pressure probably contributed to the occurrence of L-RB.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-18793685/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.virusres.2008.08.010" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2008">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Alla G Gagarinova</span>, <span >Mohan Babu</span>, <span >Martina V Strömvik</span>, <span >Aiming Wang</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November 2008
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Virol. J.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-19036160/">Recombination analysis of Soybean mosaic virus sequences reveals evidence of RNA recombination between distinct pathotypes.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-19036160/" class="summary-link">
    <div class="article-style">
      <p>RNA recombination is one of the two major factors that create RNA genome variability. Assessing its incidence in plant RNA viruses helps understand the formation of new isolates and evaluate the effectiveness of crop protection strategies. To search for recombination in Soybean mosaic virus (SMV), the causal agent of a worldwide seed-borne, aphid-transmitted viral soybean disease, we obtained all full-length genome sequences of SMV as well as partial sequences encoding the N-terminal most (P1 protease) and the C-terminal most (capsid protein; CP) viral protein. The sequences were analyzed for possible recombination events using a variety of automatic and manual recombination detection and verification approaches. Automatic scanning identified 3, 10, and 17 recombination sites in the P1, CP, and full-length sequences, respectively. Manual analyses confirmed 10 recombination sites in three full-length SMV sequences. To our knowledge, this is the first report of recombination between distinct SMV pathotypes. These data imply that different SMV pathotypes can simultaneously infect a host cell and exchange genetic materials through recombination. The high incidence of SMV recombination suggests that recombination plays an important role in SMV evolution. Obtaining additional full-length sequences will help elucidate this role.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-19036160/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/1743-422X-5-143" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2008">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Gareth Butland</span>, <span >Mohan Babu</span>, <span >J Javier Díaz-Mejía</span>, <span >Fedyshyn Bohdana</span>, <span >Sadhna Phanse</span>, <span >Barbara Gold</span>, <span >Wenhong Yang</span>, <span >Joyce Li</span>, <span >Alla G Gagarinova</span>, <span >Oxana Pogoutse</span>, <span >Hirotada Mori</span>, <span >Barry L Wanner</span>, <span >Henry Lo</span>, <span >Jas Wasniewski</span>, <span >Constantine Christopolous</span>, <span >Mehrab Ali</span>, <span >Pascal Venn</span>, <span >Anahita Safavi-Naini</span>, <span >Natalie Sourour</span>, <span >Simone Caron</span>, <span >Ja-Yeon Choi</span>, <span >Ludovic Laigle</span>, <span >Anaies Nazarians-Armavil</span>, <span >Avnish Deshpande</span>, <span >Sarah Joe</span>, <span >Kirill A Datsenko</span>, <span >Natsuko Yamamoto</span>, <span >Brenda J Andrews</span>, <span >Charles Boone</span>, <span >Huiming Ding</span>, <span >Bilal Sheikh</span>, <span >Gabriel Moreno-Hagelseib</span>, <span >Jack F Greenblatt</span>, <span >Andrew Emili</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2008
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat. Methods</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/BabuLab/publication/pmid-18677321/">
      <img src="/BabuLab/publication/pmid-18677321/featured_hue4c65b970e9ca403614850b15859488e_85536_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="eSGA: E. coli synthetic genetic array analysis.">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-18677321/">eSGA: E. coli synthetic genetic array analysis.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-18677321/" class="summary-link">
    <div class="article-style">
      <p>Physical and functional interactions define the molecular organization of the cell. Genetic interactions, or epistasis, tend to occur between gene products involved in parallel pathways or interlinked biological processes. High-throughput experimental systems to examine genetic interactions on a genome-wide scale have been devised for Saccharomyces cerevisiae, Schizosaccharomyces pombe, Caenorhabditis elegans and Drosophila melanogaster, but have not been reported previously for prokaryotes. Here we describe the development of a quantitative screening procedure for monitoring bacterial genetic interactions based on conjugation of Escherichia coli deletion or hypomorphic strains to create double mutants on a genome-wide scale. The patterns of synthetic sickness and synthetic lethality (aggravating genetic interactions) we observed for certain double mutant combinations provided information about functional relationships and redundancy between pathways and enabled us to group bacterial gene products into functional modules.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-18677321/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nmeth.1239" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2008">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Xurong Tang</span>, <span >Anfu Hou</span>, <span >Mohan Babu</span>, <span >Vi Nguyen</span>, <span >Lidia Hurtado</span>, <span >Qing Lu</span>, <span >Jose C Reyes</span>, <span >Aiming Wang</span>, <span >Wilfred A Keller</span>, <span >John J Harada</span>, <span >Edward W T Tsang</span>, <span >Yuhai Cui</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2008
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Plant Physiol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-18508955/">The Arabidopsis BRAHMA chromatin-remodeling ATPase is involved in repression of seed maturation genes in leaves.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-18508955/" class="summary-link">
    <div class="article-style">
      <p>Synthesis and accumulation of seed storage proteins (SSPs) is an important aspect of the seed maturation program. Genes encoding SSPs are specifically and highly expressed in the seed during maturation. However, the mechanisms that repress the expression of these genes in leaf tissue are not well understood. To gain insight into the repression mechanisms, we performed a genetic screen for mutants that express SSPs in leaves. Here, we show that mutations affecting BRAHMA (BRM), a SNF2 chromatin-remodeling ATPase, cause ectopic expression of a subset of SSPs and other embryogenesis-related genes in leaf tissue. Consistent with the notion that such SNF2-like ATPases form protein complexes in vivo, we observed similar phenotypes for mutations of AtSWI3C, a BRM-interacting partner, and BSH, a SNF5 homolog and essential SWI/SNF subunit. Chromatin immunoprecipitation experiments show that BRM is recruited to the promoters of a number of embryogenesis genes in wild-type leaves, including the 2S genes, expressed in brm leaves. Consistent with its role in nucleosome remodeling, BRM appears to affect the chromatin structure of the At2S2 promoter. Thus, the BRM-containing chromatin-remodeling ATPase complex involved in many aspects of plant development mediates the repression of SSPs in leaf tissue.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-18508955/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1104/pp.108.121996" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2008">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Jonathan S Griffiths</span>, <span >Tyng-Shyan Huang</span>, <span >Aiming Wang</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2008
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Genomics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-18613973/">Altered gene expression changes in Arabidopsis leaf tissues and protoplasts in response to Plum pox virus infection.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-18613973/" class="summary-link">
    <div class="article-style">
      <p>Virus infection induces the activation and suppression of global gene expression in the host. Profiling gene expression changes in the host may provide insights into the molecular mechanisms that underlie host physiological and phenotypic responses to virus infection. In this study, the Arabidopsis Affymetrix ATH1 array was used to assess global gene expression changes in Arabidopsis thaliana plants infected with Plum pox virus (PPV). To identify early genes in response to PPV infection, an Arabidopsis synchronized single-cell transformation system was developed. Arabidopsis protoplasts were transfected with a PPV infectious clone and global gene expression changes in the transfected protoplasts were profiled.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-18613973/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/1471-2164-9-325" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2008">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan Babu</span>, <span >Alla G Gagarinova</span>, <span >James E Brandle</span>, <span >Aiming Wang</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2008
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Gen. Virol.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-18343851/">Association of the transcriptional response of soybean plants with soybean mosaic virus systemic infection.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-18343851/" class="summary-link">
    <div class="article-style">
      <p>Compatible virus infection induces and suppresses host gene expression at the global level. These gene-expression changes are the molecular basis of symptom development and general stress and defence-like responses of the host. To assess transcriptional changes in soybean plants infected with soybean mosaic virus (SMV), the first soybean trifoliate leaf, immediately above the SMV-inoculated unifoliate leaf, was sampled at 7, 14 and 21 days post-inoculation (p.i.) and subjected to microarray analysis. The identified changes in gene expression in soybean leaves with SMV infection at different time points were associated with the observed symptom development. By using stringent selection criteria (&gt;or=2- or &lt;or=-2-fold change and a Q value of &lt;or=0.05), 273 (1.5 %) and 173 (0.9 %) transcripts were identified to be up- and downregulated, respectively, from 18 613 soybean cDNAs on the array. The expression levels of many transcripts encoding proteins for hormone metabolism, cell-wall biogenesis, chloroplast functions and photosynthesis were repressed at 14 days p.i. and were associated with the highest levels of viral RNA in the host cells. A number of transcripts corresponding to genes involved in defence were either downregulated or not affected at the early stages of infection, but upregulated at the late stages, indicating that the plant immune response is not activated until the late time points of infection. Such a delayed defence response may be critical for SMV to establish its systemic infection.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-18343851/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1099/vir.0.83531-0" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2005">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ali R Zahiri</span>, <span >Mohan R Babu</span>, <span >Barry J Saville</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June 2005
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Mol. Genet. Genomics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-15887033/">Differential gene expression during teliospore germination in Ustilago maydis.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-15887033/" class="summary-link">
    <div class="article-style">
      <p>Ustilago maydis is a model fungal pathogen that induces the formation of tumors in maize. The tumor provides an environment for hyphal differentiation, leading to the formation of thick-walled, diploid teliospores. Such spores serve as a dispersal agent for smut and rust fungi, and their germination leads to new rounds of infection. The morphological changes that occur during teliospore germination in U. maydis have been described in detail. However, the specific molecular events that facilitate this process have not been identified. Through the construction and hybridization of microarrays containing a set of 3918 non-redundant cDNAs, we have identified genes that are differentially regulated during teliospore germination. Teliospores induced to germinate for 4 and 11 h were selected for comparison with dormant teliospores. Genes identified as differentially expressed included many that are presumably involved in as yet undescribed molecular events during teliospore germination, as well as characterized genes previously shown to be required for the process. This study represents the first large-scale investigation of changes in gene expression during teliospore germination.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-15887033/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/s00438-005-1142-9" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2005">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Mohan R Babu</span>, <span >Kristen Choffe</span>, <span >Barry J Saville</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May 2005
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Curr. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-15809875/">Differential gene expression in filamentous cells of Ustilago maydis.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-15809875/" class="summary-link">
    <div class="article-style">
      <p>When fungi interact with plants as pathogens or as symbionts, there are often changes in fungal cell morphology and nuclear state. This study establishes the use of cDNA microarrays to detect gene expression changes in Ustilago maydis cells that differ in structure and nuclear content. Categorizing differentially expressed genes on the basis of function indicated that U. maydis cell types vary most in the expression of genes related to metabolism. We also observed that more genes are up-regulated in the filamentous dikaryon than in the filamentous diploid, relative to non-pathogenic budding cells. Our comparison of pathogenic development indicated that the dikaryon is more virulent than the diploid. Other identified expression patterns suggest a cell-specific difference in nutrient acquisition, cell metabolism and signal transduction. The relevance of gene expression change to cell type biology is discussed.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-15809875/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/s00294-005-0574-4" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2004">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ryan Austin</span>, <span >Nicholas J Provart</span>, <span >Nuno T Sacadura</span>, <span >Kimberly G Nugent</span>, <span >Mohan Babu</span>, <span >Barry J Saville</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October 2004
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Funct. Integr. Genomics</em>
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/BabuLab/publication/pmid-15349794/">A comparative genomic analysis of ESTs from Ustilago maydis.</a>
  </h3>

  
  <a href="/BabuLab/publication/pmid-15349794/" class="summary-link">
    <div class="article-style">
      <p>A large-scale comparative genomic analysis of unisequence sets obtained from an Ustilago maydis EST collection was performed against publicly available EST and genomic sequence datasets from 21 species. We annotated 70% of the collection based on similarity to known sequences and recognized protein signatures. Distinct grouping of the ESTs, defined by the presence or absence of similar sequences in the species examined, allowed the identification of U. maydis sequences present only (1) in fungal species, (2) in plants but not animals, (3) in animals but not plants, or (4) in all three eukaryotic lineages assessed. We also identified 215 U. maydis genes that are found in the ascomycete but not in the basidiomycete genome sequences searched. Candidate genes were identified for further functional characterization. These include 167 basidiomycete-specific sequences, 58 fungal pathogen-specific sequences (including 37 basidiomycete pathogen-specific sequences), and 18 plant pathogen-specific sequences, as well as two sequences present only in other plant pathogen and plant species.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/BabuLab/publication/pmid-15349794/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/s10142-004-0118-x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        
      </div>

    </div>
  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js" integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js"></script>
        
      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js" integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin="anonymous"></script>
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/BabuLab/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/BabuLab/js/academic.min.66c553246b0f279a03be6e5597f72b52.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    <a href = "https://www.uregina.ca/contact/privacy-legal/">University of Regina Web Privacy and Legal Disclaimer</a>
  </p>
  <p class="powered-by">
    © 2020
  </p>

  
  






  <p class="powered-by">
    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>
</body>
</html>
